 <div class="container_24"><div class="grid_24 clearfix" id="content"><div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp1328_c0_seq3|m.5893 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">421 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




          <h3>Protein family membership</h3>
          <div class="Family rel_tree"><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR002259">Equilibrative nucleoside transporter</a><span> (IPR002259)</span></li></ul></div>



    <h3>Domains and repeats</h3>


    <div class="prot_sum" style="background:none;">
    <div class="bot-row">
        None predicted.
    </div>
    </div>


    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Family-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_family_small.png"
                                                         alt="Family" title="Family"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR002259">IPR002259</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR002259"
                                                         class="neutral">Equilibrative nucleoside transporter</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR002259-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pir.georgetown.edu/cgi-bin/ipcSF?id=PIRSF016379" title="PIRSF016379 (ENT)"
       class="neutral">
        PIRSF016379
</a>

                <span>(ENT)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PIRSF016379-span-1"
      class="match pirsf Family"
      style="left:  0%;
              width: 99.525%;"
      title="PIRSF016379 1 - 419" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PIRSF016379-popup-1" style="display: none;">

        <div class="popup_topl"><span class="pirsf Family caption_puce"></span>1 - 419</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PIRSF</b> <abbr class="icon icon-generic" data-icon="i" title="PIRSF is a hierarchical classification system based on the 'homeomorphic family' principle. Members are both homologous (evolved from a common ancestor) and homeomorphic (sharing full-length sequence similarity and common domain architecture). PIRSF subfamilies are clusters representing functional specialization and/or domain architecture variation within the family."></abbr> <br/>
            <a href='http://pir.georgetown.edu/cgi-bin/ipcSF?id=PIRSF016379' class="ext">PIRSF016379 </a>
            <span>(ENT)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:11.876%;" title="50"></span>
            <span class="grade" style="left:23.753%;" title="100"></span>
            <span class="grade" style="left:35.629%;" title="150"></span>
            <span class="grade" style="left:47.506%;" title="200"></span>
            <span class="grade" style="left:59.382%;" title="250"></span>
            <span class="grade" style="left:71.259%;" title="300"></span>
            <span class="grade" style="left:83.135%;" title="350"></span>
            <span class="grade" style="left:100%;" title="421"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR002259-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01130&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR01130 (DERENTRNSPRT)"
       class="neutral">
        PR01130
</a>

                <span>(DERENTRNSPRT)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR01130-span-2"
      class="match prints Family"
      style="left:  65.558%;
              width: 4.276%;"
      title="PR01130 277 - 294" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01130-popup-2" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>277 - 294</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01130&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01130 </a>
            <span>(DERENTRNSPRT)</span>
        </div>
    </div>


<span id="PR01130-span-3"
      class="match prints Family"
      style="left:  30.404%;
              width: 5.463%;"
      title="PR01130 129 - 151" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01130-popup-3" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>129 - 151</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01130&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01130 </a>
            <span>(DERENTRNSPRT)</span>
        </div>
    </div>


<span id="PR01130-span-4"
      class="match prints Family"
      style="left:  78.86%;
              width: 4.276%;"
      title="PR01130 333 - 350" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01130-popup-4" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>333 - 350</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01130&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01130 </a>
            <span>(DERENTRNSPRT)</span>
        </div>
    </div>


<span id="PR01130-span-5"
      class="match prints Family"
      style="left:  37.055%;
              width: 4.988%;"
      title="PR01130 157 - 177" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01130-popup-5" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>157 - 177</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01130&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01130 </a>
            <span>(DERENTRNSPRT)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:11.876%;" title="50"></span>
            <span class="grade" style="left:23.753%;" title="100"></span>
            <span class="grade" style="left:35.629%;" title="150"></span>
            <span class="grade" style="left:47.506%;" title="200"></span>
            <span class="grade" style="left:59.382%;" title="250"></span>
            <span class="grade" style="left:71.259%;" title="300"></span>
            <span class="grade" style="left:83.135%;" title="350"></span>
            <span class="grade" style="left:100%;" title="421"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR002259-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF01733" title="PF01733 (Nucleoside_tran)"
       class="neutral">
        PF01733
</a>

            <span>(Nucleoside...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF01733-span-6"
      class="match pfam Family"
      style="left:  29.454%;
              width: 69.121%;"
      title="PF01733 125 - 415" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF01733-popup-6" style="display: none;">

        <div class="popup_topl"><span class="pfam Family caption_puce"></span>125 - 415</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF01733' class="ext">PF01733 </a>
            <span>(Nucleoside_tran)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:11.876%;" title="50"></span>
            <span class="grade" style="left:23.753%;" title="100"></span>
            <span class="grade" style="left:35.629%;" title="150"></span>
            <span class="grade" style="left:47.506%;" title="200"></span>
            <span class="grade" style="left:59.382%;" title="250"></span>
            <span class="grade" style="left:71.259%;" title="300"></span>
            <span class="grade" style="left:83.135%;" title="350"></span>
            <span class="grade" style="left:100%;" title="421"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>



<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p>
            <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810"
                   class="ext">GO:0006810</a> transport
                <br/>
            </p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005337"
                   class="ext">GO:0005337</a> nucleoside transmembrane transporter activity
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p>
                <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021"
                   class="ext">GO:0016021</a> integral component of membrane
                <br/>
            </p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 0);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
