 <div class="container_24"><div class="grid_24 clearfix" id="content"><div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp1164_c0_seq1|m.5107 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">1,536 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




        <h3>Protein family membership</h3>
        <p>None predicted.</p>



    <h3>Domains and repeats</h3>


    <div class="prot_sum">
    <div class="bot-row">
        <div class="bot-row-line-top"></div>
        <ol class="signatures">



    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-1"
      class="match c4 Domain"
      style="left:  2.018%;
              width: 49.023%;"
      title="Domain 32 - 784" >
</span>
        <div id="supermatch-popup-1" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>32 - 784
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR001609">Myosin head, motor domain</a><span> (IPR001609)</span></li></ul></li></ul></div>
    </div>




<span id="supermatch-span-2"
      class="match c4 Domain"
      style="left:  50.977%;
              width: 6.445%;"
      title="Domain 784 - 882" >
</span>
        <div id="supermatch-popup-2" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>784 - 882
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span></li></ul></div>
    </div>




<span id="supermatch-span-3"
      class="match c59 Domain"
      style="left:  75.781%;
              width: 20.508%;"
      title="Domain 1,165 - 1,479" >
</span>
        <div id="supermatch-popup-3" style="display: none;">
        <div class="popup_topl">
            <span class="c59 Domain caption_puce"></span>1,165 - 1,479
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR002710">Dilute domain</a><span> (IPR002710)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:13.021%;" title="200"></span>
                    <span class="grade" style="left:26.042%;" title="400"></span>
                    <span class="grade" style="left:39.062%;" title="600"></span>
                    <span class="grade" style="left:52.083%;" title="800"></span>
                    <span class="grade" style="left:65.104%;" title="1000"></span>
                    <span class="grade" style="left:78.125%;" title="1200"></span>
                    <span class="grade" style="left:91.146%;" title="1400"></span>
                    <span class="grade" style="left:100%;" title="1536"></span>

            </div>
        </div>
    </li>
    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-4"
      class="match c50 Domain"
      style="left:  49.414%;
              width: 2.669%;"
      title="Domain 760 - 800" >
</span>
        <div id="supermatch-popup-4" style="display: none;">
        <div class="popup_topl">
            <span class="c50 Domain caption_puce"></span>760 - 800
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027401">Myosin-like IQ motif-containing domain</a><span> (IPR027401)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:13.021%;" title="200"></span>
                    <span class="grade" style="left:26.042%;" title="400"></span>
                    <span class="grade" style="left:39.062%;" title="600"></span>
                    <span class="grade" style="left:52.083%;" title="800"></span>
                    <span class="grade" style="left:65.104%;" title="1000"></span>
                    <span class="grade" style="left:78.125%;" title="1200"></span>
                    <span class="grade" style="left:91.146%;" title="1400"></span>
                    <span class="grade" style="left:100%;" title="1536"></span>

            </div>
        </div>
    </li>

        </ol>
        <div class="bot-row-line-bot"></div>
    </div>

    <div class="prot_scale">
        <div class="bot-row">

            <div class="bot-row-line">
                <div style="position:relative;">
                            <span class="scale_bar" style="left:0%;"
                                  title="1"></span>
                                <span class="scale_numb"
                                      style="left:1%;">1</span>
                            <span class="scale_bar" style="left:13.021%;"
                                  title="200"></span>
                                <span class="scale_numb"
                                      style="left:13.021%;">200</span>
                            <span class="scale_bar" style="left:26.042%;"
                                  title="400"></span>
                                <span class="scale_numb"
                                      style="left:26.042%;">400</span>
                            <span class="scale_bar" style="left:39.062%;"
                                  title="600"></span>
                                <span class="scale_numb"
                                      style="left:39.062%;">600</span>
                            <span class="scale_bar" style="left:52.083%;"
                                  title="800"></span>
                                <span class="scale_numb"
                                      style="left:52.083%;">800</span>
                            <span class="scale_bar" style="left:65.104%;"
                                  title="1000"></span>
                                <span class="scale_numb"
                                      style="left:65.104%;">1000</span>
                            <span class="scale_bar" style="left:78.125%;"
                                  title="1200"></span>
                                <span class="scale_numb"
                                      style="left:78.125%;">1200</span>
                            <span class="scale_bar" style="left:91.146%;"
                                  title="1400"></span>
                                <span class="scale_numb"
                                      style="left:91.146%;">1400</span>
                            <span class="scale_bar" style="left:100%;"
                                  title="1536"></span>
                                <span class="scale_numb"
                                      style="left:100%;">1536</span>
                </div>
            </div>

        </div>
    </div>

    </div>



    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR027417">IPR027417</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417"
                                                         class="neutral">P-loop containing nucleoside triphosphate hydrolase</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR027417-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/3.40.50.300" title="G3DSA:3.40.50.300"
           class="neutral">
    G3DSA:3.40.50...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA34050300-span-1"
      class="match gene3d c4 Domain"
      style="left:  9.245%;
              width: 1.042%;"
      title="G3DSA:3.40.50.300 143 - 158" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA34050300-popup-1" style="display: none;">

        <div class="popup_topl"><span class="gene3d c4 Domain caption_puce"></span>143 - 158</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/3.40.50.300' class="ext">G3DSA:3.40.50.300 </a>
            <span>(G3DSA:3.40.50.300)</span>
        </div>
    </div>


<span id="G3DSA34050300-span-2"
      class="match gene3d c4 Domain"
      style="left:  24.023%;
              width: 2.865%;"
      title="G3DSA:3.40.50.300 370 - 413" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA34050300-popup-2" style="display: none;">

        <div class="popup_topl"><span class="gene3d c4 Domain caption_puce"></span>370 - 413</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/3.40.50.300' class="ext">G3DSA:3.40.50.300 </a>
            <span>(G3DSA:3.40.50.300)</span>
        </div>
    </div>


<span id="G3DSA34050300-span-3"
      class="match gene3d c4 Domain"
      style="left:  15.299%;
              width: 2.865%;"
      title="G3DSA:3.40.50.300 236 - 279" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA34050300-popup-3" style="display: none;">

        <div class="popup_topl"><span class="gene3d c4 Domain caption_puce"></span>236 - 279</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/3.40.50.300' class="ext">G3DSA:3.40.50.300 </a>
            <span>(G3DSA:3.40.50.300)</span>
        </div>
    </div>


<span id="G3DSA34050300-span-4"
      class="match gene3d c4 Domain"
      style="left:  10.286%;
              width: 3.646%;"
      title="G3DSA:3.40.50.300 159 - 214" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA34050300-popup-4" style="display: none;">

        <div class="popup_topl"><span class="gene3d c4 Domain caption_puce"></span>159 - 214</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/3.40.50.300' class="ext">G3DSA:3.40.50.300 </a>
            <span>(G3DSA:3.40.50.300)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR027417-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540" title="SSF52540 (P-loop containing nucleoside triphosphate hydrolases)"
       class="neutral">
        SSF52540
</a>

            <span>(P-loop co...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF52540-span-5"
      class="match superfamily c4 Domain"
      style="left:  50.977%;
              width: 6.445%;"
      title="SSF52540 784 - 882" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-5" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>784 - 882</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


<span id="SSF52540-span-6"
      class="match superfamily c4 Domain"
      style="left:  2.018%;
              width: 49.023%;"
      title="SSF52540 32 - 784" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-6" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>32 - 784</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR001609">IPR001609</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR001609"
                                                         class="neutral">Myosin head, motor domain</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR001609-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00193&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00193 (MYOSINHEAVY)"
       class="neutral">
        PR00193
</a>

                <span>(MYOSINHEAVY)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00193-span-7"
      class="match prints c4 Domain"
      style="left:  31.185%;
              width: 1.888%;"
      title="PR00193 480 - 508" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00193-popup-7" style="display: none;">

        <div class="popup_topl"><span class="prints c4 Domain caption_puce"></span>480 - 508</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00193&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00193 </a>
            <span>(MYOSINHEAVY)</span>
        </div>
    </div>


<span id="PR00193-span-8"
      class="match prints c4 Domain"
      style="left:  5.859%;
              width: 1.302%;"
      title="PR00193 91 - 110" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00193-popup-8" style="display: none;">

        <div class="popup_topl"><span class="prints c4 Domain caption_puce"></span>91 - 110</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00193&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00193 </a>
            <span>(MYOSINHEAVY)</span>
        </div>
    </div>


<span id="PR00193-span-9"
      class="match prints c4 Domain"
      style="left:  12.63%;
              width: 1.823%;"
      title="PR00193 195 - 222" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00193-popup-9" style="display: none;">

        <div class="popup_topl"><span class="prints c4 Domain caption_puce"></span>195 - 222</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00193&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00193 </a>
            <span>(MYOSINHEAVY)</span>
        </div>
    </div>


<span id="PR00193-span-10"
      class="match prints c4 Domain"
      style="left:  27.734%;
              width: 1.888%;"
      title="PR00193 427 - 455" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00193-popup-10" style="display: none;">

        <div class="popup_topl"><span class="prints c4 Domain caption_puce"></span>427 - 455</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00193&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00193 </a>
            <span>(MYOSINHEAVY)</span>
        </div>
    </div>


<span id="PR00193-span-11"
      class="match prints c4 Domain"
      style="left:  9.57%;
              width: 1.693%;"
      title="PR00193 148 - 173" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00193-popup-11" style="display: none;">

        <div class="popup_topl"><span class="prints c4 Domain caption_puce"></span>148 - 173</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00193&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00193 </a>
            <span>(MYOSINHEAVY)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001609-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS51456" title="PS51456 (MYOSIN_MOTOR)"
       class="neutral">
        PS51456
</a>

                <span>(MYOSIN_MOTOR)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS51456-span-12"
      class="match prosite-profiles c4 Domain"
      style="left:  3.906%;
              width: 43.685%;"
      title="PS51456 61 - 731" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS51456-popup-12" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c4 Domain caption_puce"></span>61 - 731</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS51456' class="ext">PS51456 </a>
            <span>(MYOSIN_MOTOR)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001609-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00063" title="PF00063 (Myosin_head)"
       class="neutral">
        PF00063
</a>

                <span>(Myosin_head)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00063-span-13"
      class="match pfam c4 Domain"
      style="left:  4.036%;
              width: 42.773%;"
      title="PF00063 63 - 719" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00063-popup-13" style="display: none;">

        <div class="popup_topl"><span class="pfam c4 Domain caption_puce"></span>63 - 719</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00063' class="ext">PF00063 </a>
            <span>(Myosin_head)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001609-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00242&amp;BLAST=DUMMY" title="SM00242 (MYSc)"
       class="neutral">
        SM00242
</a>

                <span>(MYSc)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00242-span-14"
      class="match smart c4 Domain"
      style="left:  3.516%;
              width: 44.141%;"
      title="SM00242 55 - 732" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00242-popup-14" style="display: none;">

        <div class="popup_topl"><span class="smart c4 Domain caption_puce"></span>55 - 732</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00242&amp;BLAST=DUMMY' class="ext">SM00242 </a>
            <span>(MYSc)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR027401">IPR027401</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR027401"
                                                         class="neutral">Myosin-like IQ motif-containing domain</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR027401-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/4.10.270.10" title="G3DSA:4.10.270.10"
           class="neutral">
    G3DSA:4.10.27...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA41027010-span-15"
      class="match gene3d c50 Domain"
      style="left:  49.414%;
              width: 2.669%;"
      title="G3DSA:4.10.270.10 760 - 800" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA41027010-popup-15" style="display: none;">

        <div class="popup_topl"><span class="gene3d c50 Domain caption_puce"></span>760 - 800</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.270.10' class="ext">G3DSA:4.10.270.10 </a>
            <span>(G3DSA:4.10.270.10)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR002710">IPR002710</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR002710"
                                                         class="neutral">Dilute domain</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR002710-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF01843" title="PF01843 (DIL)"
       class="neutral">
        PF01843
</a>

                <span>(DIL)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF01843-span-16"
      class="match pfam c59 Domain"
      style="left:  88.086%;
              width: 6.836%;"
      title="PF01843 1,354 - 1,458" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF01843-popup-16" style="display: none;">

        <div class="popup_topl"><span class="pfam c59 Domain caption_puce"></span>1,354 - 1,458</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF01843' class="ext">PF01843 </a>
            <span>(DIL)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR002710-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS51126" title="PS51126 (DILUTE)"
       class="neutral">
        PS51126
</a>

                <span>(DILUTE)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS51126-span-17"
      class="match prosite-profiles c59 Domain"
      style="left:  75.781%;
              width: 20.508%;"
      title="PS51126 1,165 - 1,479" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS51126-popup-17" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c59 Domain caption_puce"></span>1,165 - 1,479</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS51126' class="ext">PS51126 </a>
            <span>(DILUTE)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry SITE-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_site_small.png"
                                                         alt="SITE" title="SITE"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR000048">IPR000048</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR000048"
                                                         class="neutral">IQ motif, EF-hand binding site</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR000048-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY" title="SM00015 (IQ)"
       class="neutral">
        SM00015
</a>

                <span>(IQ)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00015-span-18"
      class="match smart SITE"
      style="left:  52.279%;
              width: 1.497%;"
      title="SM00015 804 - 826" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-18" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>804 - 826</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-19"
      class="match smart SITE"
      style="left:  53.906%;
              width: 1.497%;"
      title="SM00015 829 - 851" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-19" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>829 - 851</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-20"
      class="match smart SITE"
      style="left:  50.781%;
              width: 1.497%;"
      title="SM00015 781 - 803" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-20" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>781 - 803</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-21"
      class="match smart SITE"
      style="left:  49.154%;
              width: 1.497%;"
      title="SM00015 756 - 778" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-21" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>756 - 778</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-22"
      class="match smart SITE"
      style="left:  47.656%;
              width: 1.497%;"
      title="SM00015 733 - 755" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-22" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>733 - 755</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-23"
      class="match smart SITE"
      style="left:  55.404%;
              width: 1.497%;"
      title="SM00015 852 - 874" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-23" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>852 - 874</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000048-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS50096" title="PS50096 (IQ)"
       class="neutral">
        PS50096
</a>

                <span>(IQ)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS50096-span-24"
      class="match prosite-profiles SITE"
      style="left:  47.721%;
              width: 1.953%;"
      title="PS50096 734 - 763" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-24" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>734 - 763</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-25"
      class="match prosite-profiles SITE"
      style="left:  55.469%;
              width: 1.953%;"
      title="PS50096 853 - 882" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-25" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>853 - 882</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-26"
      class="match prosite-profiles SITE"
      style="left:  50.846%;
              width: 1.758%;"
      title="PS50096 782 - 808" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-26" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>782 - 808</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-27"
      class="match prosite-profiles SITE"
      style="left:  49.219%;
              width: 1.953%;"
      title="PS50096 757 - 786" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-27" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>757 - 786</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-28"
      class="match prosite-profiles SITE"
      style="left:  53.971%;
              width: 1.758%;"
      title="PS50096 830 - 856" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-28" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>830 - 856</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-29"
      class="match prosite-profiles SITE"
      style="left:  52.344%;
              width: 1.953%;"
      title="PS50096 805 - 834" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-29" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>805 - 834</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000048-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00612" title="PF00612 (IQ)"
       class="neutral">
        PF00612
</a>

                <span>(IQ)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00612-span-30"
      class="match pfam SITE"
      style="left:  52.409%;
              width: 1.367%;"
      title="PF00612 806 - 826" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-30" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>806 - 826</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-31"
      class="match pfam SITE"
      style="left:  47.852%;
              width: 1.237%;"
      title="PF00612 736 - 754" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-31" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>736 - 754</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-32"
      class="match pfam SITE"
      style="left:  54.036%;
              width: 1.367%;"
      title="PF00612 831 - 851" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-32" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>831 - 851</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-33"
      class="match pfam SITE"
      style="left:  50.977%;
              width: 1.237%;"
      title="PF00612 784 - 802" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-33" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>784 - 802</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-34"
      class="match pfam SITE"
      style="left:  49.284%;
              width: 1.302%;"
      title="PF00612 758 - 777" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-34" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>758 - 777</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-35"
      class="match pfam SITE"
      style="left:  55.534%;
              width: 1.367%;"
      title="PF00612 854 - 874" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-35" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>854 - 874</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>


            <div class="prot_entries" id="uni_sign">
                <div class="top-row">
                    <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_uni_small.png"
                                                 alt="Unintegrated signatures" title="Unintegrated signatures"/> no IPR
                    </div>
                    <div class="top-row-name">Unintegrated signatures</div>
                </div>
                <div class="bot-row">
                    <div class="bot-row-line-top"></div>
                    <ol class="signatures">
                            <li class="signature">
<div class="bot-row-signame">
        <a href="http://embnet.vital-it.ch/software/COILS_form.html" title="Coil"
           class="neutral">
        Coil
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="Coil-span-36"
      class="match coils uni"
      style="left:  63.346%;
              width: 5.208%;"
      title="Coil 974 - 1,053" >
</span>

    <script type="text/javascript">

    </script>

    <div id="Coil-popup-36" style="display: none;">

        <div class="popup_topl"><span class="coils uni caption_puce"></span>974 - 1,053</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>COILS</b> <abbr class="icon icon-generic" data-icon="i" title="COILS compares a sequence to a database of known parallel two-stranded coiled-coils and derives a similarity score and then calculates the probability that the sequence will adopt a coiled-coil conformation."></abbr> <br/>
            <a href='http://embnet.vital-it.ch/software/COILS_form.html' class="ext">Coil </a>
            <span>(Coil)</span>
        </div>
    </div>


<span id="Coil-span-37"
      class="match coils uni"
      style="left:  59.505%;
              width: 2.279%;"
      title="Coil 915 - 949" >
</span>

    <script type="text/javascript">

    </script>

    <div id="Coil-popup-37" style="display: none;">

        <div class="popup_topl"><span class="coils uni caption_puce"></span>915 - 949</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>COILS</b> <abbr class="icon icon-generic" data-icon="i" title="COILS compares a sequence to a database of known parallel two-stranded coiled-coils and derives a similarity score and then calculates the probability that the sequence will adopt a coiled-coil conformation."></abbr> <br/>
            <a href='http://embnet.vital-it.ch/software/COILS_form.html' class="ext">Coil </a>
            <span>(Coil)</span>
        </div>
    </div>


<span id="Coil-span-38"
      class="match coils uni"
      style="left:  71.289%;
              width: 1.823%;"
      title="Coil 1,096 - 1,123" >
</span>

    <script type="text/javascript">

    </script>

    <div id="Coil-popup-38" style="display: none;">

        <div class="popup_topl"><span class="coils uni caption_puce"></span>1,096 - 1,123</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>COILS</b> <abbr class="icon icon-generic" data-icon="i" title="COILS compares a sequence to a database of known parallel two-stranded coiled-coils and derives a similarity score and then calculates the probability that the sequence will adopt a coiled-coil conformation."></abbr> <br/>
            <a href='http://embnet.vital-it.ch/software/COILS_form.html' class="ext">Coil </a>
            <span>(Coil)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                            </li>
                            <li class="signature">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/1.20.5.190" title="G3DSA:1.20.5.190"
           class="neutral">
        G3DSA:1.20.5.190
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA1205190-span-39"
      class="match gene3d uni"
      style="left:  52.539%;
              width: 2.734%;"
      title="G3DSA:1.20.5.190 808 - 849" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA1205190-popup-39" style="display: none;">

        <div class="popup_topl"><span class="gene3d uni caption_puce"></span>808 - 849</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.20.5.190' class="ext">G3DSA:1.20.5.190 </a>
            <span>(G3DSA:1.20.5.190)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:13.021%;" title="200"></span>
            <span class="grade" style="left:26.042%;" title="400"></span>
            <span class="grade" style="left:39.062%;" title="600"></span>
            <span class="grade" style="left:52.083%;" title="800"></span>
            <span class="grade" style="left:65.104%;" title="1000"></span>
            <span class="grade" style="left:78.125%;" title="1200"></span>
            <span class="grade" style="left:91.146%;" title="1400"></span>
            <span class="grade" style="left:100%;" title="1536"></span>

    </div>

</div>
                            </li>
                    </ol>
                    <div class="bot-row-line-bot"></div>
                </div>
            </div>


<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003774"
                   class="ext">GO:0003774</a> motor activity
                <br/>
            
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515"
                   class="ext">GO:0005515</a> protein binding
                <br/>
            
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524"
                   class="ext">GO:0005524</a> ATP binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p>
                <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016459"
                   class="ext">GO:0016459</a> myosin complex
                <br/>
            </p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 4);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
