 <div class="container_24"><div class="grid_24 clearfix" id="content"><div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp1093_c0_seq1|m.4778 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">317 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




        <h3>Protein family membership</h3>
        <p>None predicted.</p>



    <h3>Domains and repeats</h3>


    <div class="prot_sum">
    <div class="bot-row">
        <div class="bot-row-line-top"></div>
        <ol class="signatures">



    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-1"
      class="match c13 Domain"
      style="left:  0%;
              width: 41.325%;"
      title="Domain 1 - 131" >
</span>
        <div id="supermatch-popup-1" style="display: none;">
        <div class="popup_topl">
            <span class="c13 Domain caption_puce"></span>1 - 131
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR000008">C2 domain</a><span> (IPR000008)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:15.773%;" title="50"></span>
                    <span class="grade" style="left:31.546%;" title="100"></span>
                    <span class="grade" style="left:47.319%;" title="150"></span>
                    <span class="grade" style="left:63.091%;" title="200"></span>
                    <span class="grade" style="left:78.864%;" title="250"></span>
                    <span class="grade" style="left:100%;" title="317"></span>

            </div>
        </div>
    </li>

        </ol>
        <div class="bot-row-line-bot"></div>
    </div>

    <div class="prot_scale">
        <div class="bot-row">

            <div class="bot-row-line">
                <div style="position:relative;">
                            <span class="scale_bar" style="left:0%;"
                                  title="1"></span>
                                <span class="scale_numb"
                                      style="left:1%;">1</span>
                            <span class="scale_bar" style="left:15.773%;"
                                  title="50"></span>
                                <span class="scale_numb"
                                      style="left:15.773%;">50</span>
                            <span class="scale_bar" style="left:31.546%;"
                                  title="100"></span>
                                <span class="scale_numb"
                                      style="left:31.546%;">100</span>
                            <span class="scale_bar" style="left:47.319%;"
                                  title="150"></span>
                                <span class="scale_numb"
                                      style="left:47.319%;">150</span>
                            <span class="scale_bar" style="left:63.091%;"
                                  title="200"></span>
                                <span class="scale_numb"
                                      style="left:63.091%;">200</span>
                            <span class="scale_bar" style="left:78.864%;"
                                  title="250"></span>
                                <span class="scale_numb"
                                      style="left:78.864%;">250</span>
                            <span class="scale_bar" style="left:100%;"
                                  title="317"></span>
                                <span class="scale_numb"
                                      style="left:100%;">317</span>
                </div>
            </div>

        </div>
    </div>

    </div>



    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR000008">IPR000008</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR000008"
                                                         class="neutral">C2 domain</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR000008-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00168" title="PF00168 (C2)"
       class="neutral">
        PF00168
</a>

                <span>(C2)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00168-span-1"
      class="match pfam c13 Domain"
      style="left:  3.47%;
              width: 31.23%;"
      title="PF00168 12 - 110" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00168-popup-1" style="display: none;">

        <div class="popup_topl"><span class="pfam c13 Domain caption_puce"></span>12 - 110</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00168' class="ext">PF00168 </a>
            <span>(C2)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.773%;" title="50"></span>
            <span class="grade" style="left:31.546%;" title="100"></span>
            <span class="grade" style="left:47.319%;" title="150"></span>
            <span class="grade" style="left:63.091%;" title="200"></span>
            <span class="grade" style="left:78.864%;" title="250"></span>
            <span class="grade" style="left:100%;" title="317"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000008-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/2.60.40.150" title="G3DSA:2.60.40.150"
           class="neutral">
    G3DSA:2.60.40...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA26040150-span-2"
      class="match gene3d c13 Domain"
      style="left:  3.47%;
              width: 37.224%;"
      title="G3DSA:2.60.40.150 12 - 129" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA26040150-popup-2" style="display: none;">

        <div class="popup_topl"><span class="gene3d c13 Domain caption_puce"></span>12 - 129</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/2.60.40.150' class="ext">G3DSA:2.60.40.150 </a>
            <span>(G3DSA:2.60.40.150)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.773%;" title="50"></span>
            <span class="grade" style="left:31.546%;" title="100"></span>
            <span class="grade" style="left:47.319%;" title="150"></span>
            <span class="grade" style="left:63.091%;" title="200"></span>
            <span class="grade" style="left:78.864%;" title="250"></span>
            <span class="grade" style="left:100%;" title="317"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000008-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS50004" title="PS50004 (C2)"
       class="neutral">
        PS50004
</a>

                <span>(C2)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS50004-span-3"
      class="match prosite-profiles c13 Domain"
      style="left:  0%;
              width: 29.653%;"
      title="PS50004 1 - 94" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50004-popup-3" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c13 Domain caption_puce"></span>1 - 94</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50004' class="ext">PS50004 </a>
            <span>(C2)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.773%;" title="50"></span>
            <span class="grade" style="left:31.546%;" title="100"></span>
            <span class="grade" style="left:47.319%;" title="150"></span>
            <span class="grade" style="left:63.091%;" title="200"></span>
            <span class="grade" style="left:78.864%;" title="250"></span>
            <span class="grade" style="left:100%;" title="317"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000008-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF49562" title="SSF49562 (C2 domain (Calcium/lipid-binding domain, CaLB))"
       class="neutral">
        SSF49562
</a>

            <span>(C2 domain...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF49562-span-4"
      class="match superfamily c13 Domain"
      style="left:  2.839%;
              width: 38.486%;"
      title="SSF49562 10 - 131" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF49562-popup-4" style="display: none;">

        <div class="popup_topl"><span class="superfamily c13 Domain caption_puce"></span>10 - 131</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF49562' class="ext">SSF49562 </a>
            <span>(C2 domain (Calcium/lipid-binding domain, CaLB))</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.773%;" title="50"></span>
            <span class="grade" style="left:31.546%;" title="100"></span>
            <span class="grade" style="left:47.319%;" title="150"></span>
            <span class="grade" style="left:63.091%;" title="200"></span>
            <span class="grade" style="left:78.864%;" title="250"></span>
            <span class="grade" style="left:100%;" title="317"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000008-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00239&amp;BLAST=DUMMY" title="SM00239 (C2)"
       class="neutral">
        SM00239
</a>

                <span>(C2)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00239-span-5"
      class="match smart c13 Domain"
      style="left:  3.47%;
              width: 30.915%;"
      title="SM00239 12 - 109" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00239-popup-5" style="display: none;">

        <div class="popup_topl"><span class="smart c13 Domain caption_puce"></span>12 - 109</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00239&amp;BLAST=DUMMY' class="ext">SM00239 </a>
            <span>(C2)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.773%;" title="50"></span>
            <span class="grade" style="left:31.546%;" title="100"></span>
            <span class="grade" style="left:47.319%;" title="150"></span>
            <span class="grade" style="left:63.091%;" title="200"></span>
            <span class="grade" style="left:78.864%;" title="250"></span>
            <span class="grade" style="left:100%;" title="317"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>


            <div class="prot_entries" id="uni_sign">
                <div class="top-row">
                    <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_uni_small.png"
                                                 alt="Unintegrated signatures" title="Unintegrated signatures"/> no IPR
                    </div>
                    <div class="top-row-name">Unintegrated signatures</div>
                </div>
                <div class="bot-row">
                    <div class="bot-row-line-top"></div>
                    <ol class="signatures">
                            <li class="signature">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01217&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR01217 (PRICHEXTENSN)"
       class="neutral">
        PR01217
</a>

                <span>(PRICHEXTENSN)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR01217-span-6"
      class="match prints uni"
      style="left:  46.688%;
              width: 4.101%;"
      title="PR01217 149 - 161" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01217-popup-6" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>149 - 161</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01217&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01217 </a>
            <span>(PRICHEXTENSN)</span>
        </div>
    </div>


<span id="PR01217-span-7"
      class="match prints uni"
      style="left:  53.312%;
              width: 5.363%;"
      title="PR01217 170 - 186" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01217-popup-7" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>170 - 186</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01217&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01217 </a>
            <span>(PRICHEXTENSN)</span>
        </div>
    </div>


<span id="PR01217-span-8"
      class="match prints uni"
      style="left:  71.924%;
              width: 5.363%;"
      title="PR01217 229 - 245" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01217-popup-8" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>229 - 245</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01217&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01217 </a>
            <span>(PRICHEXTENSN)</span>
        </div>
    </div>


<span id="PR01217-span-9"
      class="match prints uni"
      style="left:  60.252%;
              width: 4.101%;"
      title="PR01217 192 - 204" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01217-popup-9" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>192 - 204</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01217&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01217 </a>
            <span>(PRICHEXTENSN)</span>
        </div>
    </div>


<span id="PR01217-span-10"
      class="match prints uni"
      style="left:  64.669%;
              width: 6.94%;"
      title="PR01217 206 - 227" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01217-popup-10" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>206 - 227</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01217&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01217 </a>
            <span>(PRICHEXTENSN)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.773%;" title="50"></span>
            <span class="grade" style="left:31.546%;" title="100"></span>
            <span class="grade" style="left:47.319%;" title="150"></span>
            <span class="grade" style="left:63.091%;" title="200"></span>
            <span class="grade" style="left:78.864%;" title="250"></span>
            <span class="grade" style="left:100%;" title="317"></span>

    </div>

</div>
                            </li>
                    </ol>
                    <div class="bot-row-line-bot"></div>
                </div>
            </div>


<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515"
                   class="ext">GO:0005515</a> protein binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 1);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
