 <div class="container_24"><div class="grid_24 clearfix" id="content"><div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp2016_c0_seq3|m.8926 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">228 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




        <h3>Protein family membership</h3>
        <p>None predicted.</p>



    <h3>Domains and repeats</h3>


    <div class="prot_sum">
    <div class="bot-row">
        <div class="bot-row-line-top"></div>
        <ol class="signatures">



    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-1"
      class="match c61 Domain"
      style="left:  6.579%;
              width: 83.333%;"
      title="Domain 16 - 205" >
</span>
        <div id="supermatch-popup-1" style="display: none;">
        <div class="popup_topl">
            <span class="c61 Domain caption_puce"></span>16 - 205
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR015916">Galactose oxidase, beta-propeller</a><span> (IPR015916)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:21.93%;" title="50"></span>
                    <span class="grade" style="left:43.86%;" title="100"></span>
                    <span class="grade" style="left:65.789%;" title="150"></span>
                    <span class="grade" style="left:87.719%;" title="200"></span>
                    <span class="grade" style="left:100%;" title="228"></span>

            </div>
        </div>
    </li>
    <li class="signature entry-signatures">
        <div class="bot-row-signame">Repeat</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-2"
      class="match c42 Repeat"
      style="left:  5.263%;
              width: 59.211%;"
      title="Repeat 13 - 147" >
</span>
        <div id="supermatch-popup-2" style="display: none;">
        <div class="popup_topl">
            <span class="c42 Repeat caption_puce"></span>13 - 147
        </div>
        <div class="rel_tree"><ul><li class="Repeat" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR006652">Kelch repeat type 1</a><span> (IPR006652)</span></li></ul></div>
    </div>




<span id="supermatch-span-3"
      class="match c42 Repeat"
      style="left:  60.088%;
              width: 25%;"
      title="Repeat 138 - 194" >
</span>
        <div id="supermatch-popup-3" style="display: none;">
        <div class="popup_topl">
            <span class="c42 Repeat caption_puce"></span>138 - 194
        </div>
        <div class="rel_tree"><ul><li class="Repeat" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR006652">Kelch repeat type 1</a><span> (IPR006652)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:21.93%;" title="50"></span>
                    <span class="grade" style="left:43.86%;" title="100"></span>
                    <span class="grade" style="left:65.789%;" title="150"></span>
                    <span class="grade" style="left:87.719%;" title="200"></span>
                    <span class="grade" style="left:100%;" title="228"></span>

            </div>
        </div>
    </li>

        </ol>
        <div class="bot-row-line-bot"></div>
    </div>

    <div class="prot_scale">
        <div class="bot-row">

            <div class="bot-row-line">
                <div style="position:relative;">
                            <span class="scale_bar" style="left:0%;"
                                  title="1"></span>
                                <span class="scale_numb"
                                      style="left:1%;">1</span>
                            <span class="scale_bar" style="left:21.93%;"
                                  title="50"></span>
                                <span class="scale_numb"
                                      style="left:21.93%;">50</span>
                            <span class="scale_bar" style="left:43.86%;"
                                  title="100"></span>
                                <span class="scale_numb"
                                      style="left:43.86%;">100</span>
                            <span class="scale_bar" style="left:65.789%;"
                                  title="150"></span>
                                <span class="scale_numb"
                                      style="left:65.789%;">150</span>
                            <span class="scale_bar" style="left:87.719%;"
                                  title="200"></span>
                                <span class="scale_numb"
                                      style="left:87.719%;">200</span>
                            <span class="scale_bar" style="left:100%;"
                                  title="228"></span>
                                <span class="scale_numb"
                                      style="left:100%;">228</span>
                </div>
            </div>

        </div>
    </div>

    </div>



    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR015916">IPR015916</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR015916"
                                                         class="neutral">Galactose oxidase, beta-propeller</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR015916-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/2.130.10.80" title="G3DSA:2.130.10.80"
           class="neutral">
    G3DSA:2.130.1...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA21301080-span-1"
      class="match gene3d c61 Domain"
      style="left:  6.579%;
              width: 83.333%;"
      title="G3DSA:2.130.10.80 16 - 205" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA21301080-popup-1" style="display: none;">

        <div class="popup_topl"><span class="gene3d c61 Domain caption_puce"></span>16 - 205</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/2.130.10.80' class="ext">G3DSA:2.130.10.80 </a>
            <span>(G3DSA:2.130.10.80)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:21.93%;" title="50"></span>
            <span class="grade" style="left:43.86%;" title="100"></span>
            <span class="grade" style="left:65.789%;" title="150"></span>
            <span class="grade" style="left:87.719%;" title="200"></span>
            <span class="grade" style="left:100%;" title="228"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Repeat-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_repeat_small.png"
                                                         alt="Repeat" title="Repeat"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR006652">IPR006652</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR006652"
                                                         class="neutral">Kelch repeat type 1</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR006652-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF01344" title="PF01344 (Kelch_1)"
       class="neutral">
        PF01344
</a>

                <span>(Kelch_1)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF01344-span-2"
      class="match pfam c42 Repeat"
      style="left:  60.088%;
              width: 18.86%;"
      title="PF01344 138 - 180" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF01344-popup-2" style="display: none;">

        <div class="popup_topl"><span class="pfam c42 Repeat caption_puce"></span>138 - 180</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF01344' class="ext">PF01344 </a>
            <span>(Kelch_1)</span>
        </div>
    </div>


<span id="PF01344-span-3"
      class="match pfam c42 Repeat"
      style="left:  18.86%;
              width: 18.86%;"
      title="PF01344 44 - 86" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF01344-popup-3" style="display: none;">

        <div class="popup_topl"><span class="pfam c42 Repeat caption_puce"></span>44 - 86</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF01344' class="ext">PF01344 </a>
            <span>(Kelch_1)</span>
        </div>
    </div>


<span id="PF01344-span-4"
      class="match pfam c42 Repeat"
      style="left:  39.035%;
              width: 19.737%;"
      title="PF01344 90 - 134" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF01344-popup-4" style="display: none;">

        <div class="popup_topl"><span class="pfam c42 Repeat caption_puce"></span>90 - 134</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF01344' class="ext">PF01344 </a>
            <span>(Kelch_1)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:21.93%;" title="50"></span>
            <span class="grade" style="left:43.86%;" title="100"></span>
            <span class="grade" style="left:65.789%;" title="150"></span>
            <span class="grade" style="left:87.719%;" title="200"></span>
            <span class="grade" style="left:100%;" title="228"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR006652-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00612&amp;BLAST=DUMMY" title="SM00612 (Kelch)"
       class="neutral">
        SM00612
</a>

                <span>(Kelch)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00612-span-5"
      class="match smart c42 Repeat"
      style="left:  5.263%;
              width: 17.982%;"
      title="SM00612 13 - 53" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00612-popup-5" style="display: none;">

        <div class="popup_topl"><span class="smart c42 Repeat caption_puce"></span>13 - 53</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00612&amp;BLAST=DUMMY' class="ext">SM00612 </a>
            <span>(Kelch)</span>
        </div>
    </div>


<span id="SM00612-span-6"
      class="match smart c42 Repeat"
      style="left:  64.474%;
              width: 20.614%;"
      title="SM00612 148 - 194" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00612-popup-6" style="display: none;">

        <div class="popup_topl"><span class="smart c42 Repeat caption_puce"></span>148 - 194</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00612&amp;BLAST=DUMMY' class="ext">SM00612 </a>
            <span>(Kelch)</span>
        </div>
    </div>


<span id="SM00612-span-7"
      class="match smart c42 Repeat"
      style="left:  23.246%;
              width: 20.614%;"
      title="SM00612 54 - 100" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00612-popup-7" style="display: none;">

        <div class="popup_topl"><span class="smart c42 Repeat caption_puce"></span>54 - 100</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00612&amp;BLAST=DUMMY' class="ext">SM00612 </a>
            <span>(Kelch)</span>
        </div>
    </div>


<span id="SM00612-span-8"
      class="match smart c42 Repeat"
      style="left:  43.86%;
              width: 20.614%;"
      title="SM00612 101 - 147" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00612-popup-8" style="display: none;">

        <div class="popup_topl"><span class="smart c42 Repeat caption_puce"></span>101 - 147</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00612&amp;BLAST=DUMMY' class="ext">SM00612 </a>
            <span>(Kelch)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:21.93%;" title="50"></span>
            <span class="grade" style="left:43.86%;" title="100"></span>
            <span class="grade" style="left:65.789%;" title="150"></span>
            <span class="grade" style="left:87.719%;" title="200"></span>
            <span class="grade" style="left:100%;" title="228"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>


            <div class="prot_entries" id="uni_sign">
                <div class="top-row">
                    <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_uni_small.png"
                                                 alt="Unintegrated signatures" title="Unintegrated signatures"/> no IPR
                    </div>
                    <div class="top-row-name">Unintegrated signatures</div>
                </div>
                <div class="bot-row">
                    <div class="bot-row-line-top"></div>
                    <ol class="signatures">
                            <li class="signature">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00501&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00501 (KELCHREPEAT)"
       class="neutral">
        PR00501
</a>

                <span>(KELCHREPEAT)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00501-span-9"
      class="match prints uni"
      style="left:  42.982%;
              width: 6.14%;"
      title="PR00501 99 - 112" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00501-popup-9" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>99 - 112</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00501&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00501 </a>
            <span>(KELCHREPEAT)</span>
        </div>
    </div>


<span id="PR00501-span-10"
      class="match prints uni"
      style="left:  57.018%;
              width: 5.702%;"
      title="PR00501 131 - 143" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00501-popup-10" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>131 - 143</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00501&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00501 </a>
            <span>(KELCHREPEAT)</span>
        </div>
    </div>


<span id="PR00501-span-11"
      class="match prints uni"
      style="left:  50.439%;
              width: 6.579%;"
      title="PR00501 116 - 130" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00501-popup-11" style="display: none;">

        <div class="popup_topl"><span class="prints uni caption_puce"></span>116 - 130</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00501&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00501 </a>
            <span>(KELCHREPEAT)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:21.93%;" title="50"></span>
            <span class="grade" style="left:43.86%;" title="100"></span>
            <span class="grade" style="left:65.789%;" title="150"></span>
            <span class="grade" style="left:87.719%;" title="200"></span>
            <span class="grade" style="left:100%;" title="228"></span>

    </div>

</div>
                            </li>
                            <li class="signature">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF117281" title="SSF117281 (Kelch motif)"
       class="neutral">
        SSF117281
</a>

                <span>(Kelch motif)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF117281-span-12"
      class="match superfamily uni"
      style="left:  3.509%;
              width: 87.719%;"
      title="SSF117281 9 - 208" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF117281-popup-12" style="display: none;">

        <div class="popup_topl"><span class="superfamily uni caption_puce"></span>9 - 208</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF117281' class="ext">SSF117281 </a>
            <span>(Kelch motif)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:21.93%;" title="50"></span>
            <span class="grade" style="left:43.86%;" title="100"></span>
            <span class="grade" style="left:65.789%;" title="150"></span>
            <span class="grade" style="left:87.719%;" title="200"></span>
            <span class="grade" style="left:100%;" title="228"></span>

    </div>

</div>
                            </li>
                    </ol>
                    <div class="bot-row-line-bot"></div>
                </div>
            </div>


<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515"
                   class="ext">GO:0005515</a> protein binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 3);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
