 <div class="container_24"><div class="grid_24 clearfix" id="content"><div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp3121_c0_seq1|m.13924 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">2,111 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




        <h3>Protein family membership</h3>
        <p>None predicted.</p>



    <h3>Domains and repeats</h3>


    <div class="prot_sum">
    <div class="bot-row">
        <div class="bot-row-line-top"></div>
        <ol class="signatures">



    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-1"
      class="match c5 Domain"
      style="left:  22.975%;
              width: 16.153%;"
      title="Domain 486 - 826" >
</span>
        <div id="supermatch-popup-1" style="display: none;">
        <div class="popup_topl">
            <span class="c5 Domain caption_puce"></span>486 - 826
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR001715">Calponin homology domain</a><span> (IPR001715)</span></li></ul></div>
    </div>




<span id="supermatch-span-2"
      class="match c4 Domain"
      style="left:  57.319%;
              width: 3.742%;"
      title="Domain 1,211 - 1,289" >
</span>
        <div id="supermatch-popup-2" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>1,211 - 1,289
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span></li></ul></div>
    </div>




<span id="supermatch-span-3"
      class="match c4 Domain"
      style="left:  60.777%;
              width: 4.69%;"
      title="Domain 1,284 - 1,382" >
</span>
        <div id="supermatch-popup-3" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>1,284 - 1,382
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span></li></ul></div>
    </div>




<span id="supermatch-span-4"
      class="match c4 Domain"
      style="left:  71.767%;
              width: 2.937%;"
      title="Domain 1,516 - 1,577" >
</span>
        <div id="supermatch-popup-4" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>1,516 - 1,577
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span></li></ul></div>
    </div>




<span id="supermatch-span-5"
      class="match c4 Domain"
      style="left:  79.915%;
              width: 3.884%;"
      title="Domain 1,688 - 1,769" >
</span>
        <div id="supermatch-popup-5" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>1,688 - 1,769
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span></li></ul></div>
    </div>




<span id="supermatch-span-6"
      class="match c4 Domain"
      style="left:  94.173%;
              width: 2.842%;"
      title="Domain 1,989 - 2,048" >
</span>
        <div id="supermatch-popup-6" style="display: none;">
        <div class="popup_topl">
            <span class="c4 Domain caption_puce"></span>1,989 - 2,048
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417">P-loop containing nucleoside triphosphate hydrolase</a><span> (IPR027417)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:9.474%;" title="200"></span>
                    <span class="grade" style="left:18.948%;" title="400"></span>
                    <span class="grade" style="left:28.423%;" title="600"></span>
                    <span class="grade" style="left:37.897%;" title="800"></span>
                    <span class="grade" style="left:47.371%;" title="1000"></span>
                    <span class="grade" style="left:56.845%;" title="1200"></span>
                    <span class="grade" style="left:66.319%;" title="1400"></span>
                    <span class="grade" style="left:75.793%;" title="1600"></span>
                    <span class="grade" style="left:85.268%;" title="1800"></span>
                    <span class="grade" style="left:94.742%;" title="2000"></span>
                    <span class="grade" style="left:100%;" title="2111"></span>

            </div>
        </div>
    </li>

        </ol>
        <div class="bot-row-line-bot"></div>
    </div>

    <div class="prot_scale">
        <div class="bot-row">

            <div class="bot-row-line">
                <div style="position:relative;">
                            <span class="scale_bar" style="left:0%;"
                                  title="1"></span>
                                <span class="scale_numb"
                                      style="left:1%;">1</span>
                            <span class="scale_bar" style="left:9.474%;"
                                  title="200"></span>
                                <span class="scale_numb"
                                      style="left:9.474%;">200</span>
                            <span class="scale_bar" style="left:18.948%;"
                                  title="400"></span>
                                <span class="scale_numb"
                                      style="left:18.948%;">400</span>
                            <span class="scale_bar" style="left:28.423%;"
                                  title="600"></span>
                                <span class="scale_numb"
                                      style="left:28.423%;">600</span>
                            <span class="scale_bar" style="left:37.897%;"
                                  title="800"></span>
                                <span class="scale_numb"
                                      style="left:37.897%;">800</span>
                            <span class="scale_bar" style="left:47.371%;"
                                  title="1000"></span>
                                <span class="scale_numb"
                                      style="left:47.371%;">1000</span>
                            <span class="scale_bar" style="left:56.845%;"
                                  title="1200"></span>
                                <span class="scale_numb"
                                      style="left:56.845%;">1200</span>
                            <span class="scale_bar" style="left:66.319%;"
                                  title="1400"></span>
                                <span class="scale_numb"
                                      style="left:66.319%;">1400</span>
                            <span class="scale_bar" style="left:75.793%;"
                                  title="1600"></span>
                                <span class="scale_numb"
                                      style="left:75.793%;">1600</span>
                            <span class="scale_bar" style="left:85.268%;"
                                  title="1800"></span>
                                <span class="scale_numb"
                                      style="left:85.268%;">1800</span>
                            <span class="scale_bar" style="left:94.742%;"
                                  title="2000"></span>
                                <span class="scale_numb"
                                      style="left:94.742%;">2000</span>
                            <span class="scale_bar" style="left:100%;"
                                  title="2111"></span>
                                <span class="scale_numb"
                                      style="left:100%;">2111</span>
                </div>
            </div>

        </div>
    </div>

    </div>



    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR001715">IPR001715</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR001715"
                                                         class="neutral">Calponin homology domain</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR001715-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00307" title="PF00307 (CH)"
       class="neutral">
        PF00307
</a>

                <span>(CH)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00307-span-1"
      class="match pfam c5 Domain"
      style="left:  26.812%;
              width: 4.311%;"
      title="PF00307 567 - 657" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00307-popup-1" style="display: none;">

        <div class="popup_topl"><span class="pfam c5 Domain caption_puce"></span>567 - 657</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00307' class="ext">PF00307 </a>
            <span>(CH)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001715-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/1.10.418.10" title="G3DSA:1.10.418.10"
           class="neutral">
    G3DSA:1.10.41...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA11041810-span-2"
      class="match gene3d c5 Domain"
      style="left:  33.018%;
              width: 6.111%;"
      title="G3DSA:1.10.418.10 698 - 826" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA11041810-popup-2" style="display: none;">

        <div class="popup_topl"><span class="gene3d c5 Domain caption_puce"></span>698 - 826</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.10.418.10' class="ext">G3DSA:1.10.418.10 </a>
            <span>(G3DSA:1.10.418.10)</span>
        </div>
    </div>


<span id="G3DSA11041810-span-3"
      class="match gene3d c5 Domain"
      style="left:  26.622%;
              width: 4.879%;"
      title="G3DSA:1.10.418.10 563 - 665" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA11041810-popup-3" style="display: none;">

        <div class="popup_topl"><span class="gene3d c5 Domain caption_puce"></span>563 - 665</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.10.418.10' class="ext">G3DSA:1.10.418.10 </a>
            <span>(G3DSA:1.10.418.10)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001715-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF47576" title="SSF47576 (Calponin-homology domain, CH-domain)"
       class="neutral">
        SSF47576
</a>

            <span>(Calponin-...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF47576-span-4"
      class="match superfamily c5 Domain"
      style="left:  22.975%;
              width: 16.011%;"
      title="SSF47576 486 - 823" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF47576-popup-4" style="display: none;">

        <div class="popup_topl"><span class="superfamily c5 Domain caption_puce"></span>486 - 823</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF47576' class="ext">SSF47576 </a>
            <span>(Calponin-homology domain, CH-domain)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001715-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00033&amp;BLAST=DUMMY" title="SM00033 (CH)"
       class="neutral">
        SM00033
</a>

                <span>(CH)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00033-span-5"
      class="match smart c5 Domain"
      style="left:  24.964%;
              width: 6.348%;"
      title="SM00033 528 - 661" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00033-popup-5" style="display: none;">

        <div class="popup_topl"><span class="smart c5 Domain caption_puce"></span>528 - 661</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00033&amp;BLAST=DUMMY' class="ext">SM00033 </a>
            <span>(CH)</span>
        </div>
    </div>


<span id="SM00033-span-6"
      class="match smart c5 Domain"
      style="left:  32.875%;
              width: 6.016%;"
      title="SM00033 695 - 821" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00033-popup-6" style="display: none;">

        <div class="popup_topl"><span class="smart c5 Domain caption_puce"></span>695 - 821</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00033&amp;BLAST=DUMMY' class="ext">SM00033 </a>
            <span>(CH)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001715-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS50021" title="PS50021 (CH)"
       class="neutral">
        PS50021
</a>

                <span>(CH)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS50021-span-7"
      class="match prosite-profiles c5 Domain"
      style="left:  32.781%;
              width: 6.063%;"
      title="PS50021 693 - 820" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50021-popup-7" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c5 Domain caption_puce"></span>693 - 820</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50021' class="ext">PS50021 </a>
            <span>(CH)</span>
        </div>
    </div>


<span id="PS50021-span-8"
      class="match prosite-profiles c5 Domain"
      style="left:  24.87%;
              width: 6.537%;"
      title="PS50021 526 - 663" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50021-popup-8" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c5 Domain caption_puce"></span>526 - 663</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50021' class="ext">PS50021 </a>
            <span>(CH)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR027417">IPR027417</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR027417"
                                                         class="neutral">P-loop containing nucleoside triphosphate hydrolase</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR027417-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540" title="SSF52540 (P-loop containing nucleoside triphosphate hydrolases)"
       class="neutral">
        SSF52540
</a>

            <span>(P-loop co...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF52540-span-9"
      class="match superfamily c4 Domain"
      style="left:  71.767%;
              width: 2.937%;"
      title="SSF52540 1,516 - 1,577" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-9" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>1,516 - 1,577</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


<span id="SSF52540-span-10"
      class="match superfamily c4 Domain"
      style="left:  94.173%;
              width: 2.842%;"
      title="SSF52540 1,989 - 2,048" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-10" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>1,989 - 2,048</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


<span id="SSF52540-span-11"
      class="match superfamily c4 Domain"
      style="left:  57.319%;
              width: 3.742%;"
      title="SSF52540 1,211 - 1,289" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-11" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>1,211 - 1,289</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


<span id="SSF52540-span-12"
      class="match superfamily c4 Domain"
      style="left:  79.915%;
              width: 3.884%;"
      title="SSF52540 1,688 - 1,769" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-12" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>1,688 - 1,769</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


<span id="SSF52540-span-13"
      class="match superfamily c4 Domain"
      style="left:  60.777%;
              width: 4.69%;"
      title="SSF52540 1,284 - 1,382" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF52540-popup-13" style="display: none;">

        <div class="popup_topl"><span class="superfamily c4 Domain caption_puce"></span>1,284 - 1,382</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF52540' class="ext">SSF52540 </a>
            <span>(P-loop containing nucleoside triphosphate hydrolases)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry SITE-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="../static/interpro/resources/images/ico_type_site_small.png"
                                                         alt="SITE" title="SITE"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR000048">IPR000048</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR000048"
                                                         class="neutral">IQ motif, EF-hand binding site</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR000048-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY" title="SM00015 (IQ)"
       class="neutral">
        SM00015
</a>

                <span>(IQ)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00015-span-14"
      class="match smart SITE"
      style="left:  81.715%;
              width: 1.09%;"
      title="SM00015 1,726 - 1,748" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-14" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,726 - 1,748</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-15"
      class="match smart SITE"
      style="left:  45.381%;
              width: 1.09%;"
      title="SM00015 959 - 981" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-15" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>959 - 981</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-16"
      class="match smart SITE"
      style="left:  80.625%;
              width: 1.09%;"
      title="SM00015 1,703 - 1,725" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-16" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,703 - 1,725</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-17"
      class="match smart SITE"
      style="left:  71.577%;
              width: 1.09%;"
      title="SM00015 1,512 - 1,534" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-17" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,512 - 1,534</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-18"
      class="match smart SITE"
      style="left:  87.399%;
              width: 1.09%;"
      title="SM00015 1,846 - 1,868" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-18" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,846 - 1,868</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-19"
      class="match smart SITE"
      style="left:  82.804%;
              width: 1.09%;"
      title="SM00015 1,749 - 1,771" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-19" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,749 - 1,771</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-20"
      class="match smart SITE"
      style="left:  59.545%;
              width: 1.09%;"
      title="SM00015 1,258 - 1,280" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-20" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,258 - 1,280</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-21"
      class="match smart SITE"
      style="left:  77.357%;
              width: 1.09%;"
      title="SM00015 1,634 - 1,656" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-21" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,634 - 1,656</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-22"
      class="match smart SITE"
      style="left:  78.446%;
              width: 1.09%;"
      title="SM00015 1,657 - 1,679" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-22" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,657 - 1,679</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-23"
      class="match smart SITE"
      style="left:  61.724%;
              width: 1.09%;"
      title="SM00015 1,304 - 1,326" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-23" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,304 - 1,326</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-24"
      class="match smart SITE"
      style="left:  43.392%;
              width: 1.09%;"
      title="SM00015 917 - 939" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-24" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>917 - 939</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-25"
      class="match smart SITE"
      style="left:  68.404%;
              width: 1.042%;"
      title="SM00015 1,445 - 1,466" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-25" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,445 - 1,466</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-26"
      class="match smart SITE"
      style="left:  95.879%;
              width: 1.09%;"
      title="SM00015 2,025 - 2,047" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-26" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>2,025 - 2,047</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-27"
      class="match smart SITE"
      style="left:  55.187%;
              width: 1.09%;"
      title="SM00015 1,166 - 1,188" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-27" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,166 - 1,188</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-28"
      class="match smart SITE"
      style="left:  62.814%;
              width: 1.09%;"
      title="SM00015 1,327 - 1,349" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-28" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,327 - 1,349</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-29"
      class="match smart SITE"
      style="left:  73.757%;
              width: 1.042%;"
      title="SM00015 1,558 - 1,579" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-29" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,558 - 1,579</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-30"
      class="match smart SITE"
      style="left:  57.366%;
              width: 1.09%;"
      title="SM00015 1,212 - 1,234" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-30" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,212 - 1,234</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-31"
      class="match smart SITE"
      style="left:  64.993%;
              width: 1.09%;"
      title="SM00015 1,373 - 1,395" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-31" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,373 - 1,395</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-32"
      class="match smart SITE"
      style="left:  66.272%;
              width: 1.09%;"
      title="SM00015 1,400 - 1,422" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-32" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,400 - 1,422</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-33"
      class="match smart SITE"
      style="left:  60.635%;
              width: 1.09%;"
      title="SM00015 1,281 - 1,303" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-33" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,281 - 1,303</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-34"
      class="match smart SITE"
      style="left:  72.667%;
              width: 1.09%;"
      title="SM00015 1,535 - 1,557" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-34" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,535 - 1,557</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-35"
      class="match smart SITE"
      style="left:  56.277%;
              width: 1.09%;"
      title="SM00015 1,189 - 1,211" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-35" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,189 - 1,211</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-36"
      class="match smart SITE"
      style="left:  63.903%;
              width: 1.09%;"
      title="SM00015 1,350 - 1,372" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-36" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,350 - 1,372</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-37"
      class="match smart SITE"
      style="left:  53.198%;
              width: 1.09%;"
      title="SM00015 1,124 - 1,146" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-37" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,124 - 1,146</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-38"
      class="match smart SITE"
      style="left:  58.456%;
              width: 1.09%;"
      title="SM00015 1,235 - 1,257" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-38" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,235 - 1,257</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-39"
      class="match smart SITE"
      style="left:  67.361%;
              width: 1.042%;"
      title="SM00015 1,423 - 1,444" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-39" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,423 - 1,444</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-40"
      class="match smart SITE"
      style="left:  47.087%;
              width: 1.09%;"
      title="SM00015 995 - 1,017" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-40" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>995 - 1,017</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-41"
      class="match smart SITE"
      style="left:  94.789%;
              width: 1.09%;"
      title="SM00015 2,002 - 2,024" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-41" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>2,002 - 2,024</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-42"
      class="match smart SITE"
      style="left:  91.284%;
              width: 1.09%;"
      title="SM00015 1,928 - 1,950" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-42" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,928 - 1,950</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-43"
      class="match smart SITE"
      style="left:  75.178%;
              width: 1.09%;"
      title="SM00015 1,588 - 1,610" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-43" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,588 - 1,610</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-44"
      class="match smart SITE"
      style="left:  76.267%;
              width: 1.09%;"
      title="SM00015 1,611 - 1,633" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-44" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,611 - 1,633</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="SM00015-span-45"
      class="match smart SITE"
      style="left:  69.446%;
              width: 1.09%;"
      title="SM00015 1,467 - 1,489" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00015-popup-45" style="display: none;">

        <div class="popup_topl"><span class="smart SITE caption_puce"></span>1,467 - 1,489</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00015&amp;BLAST=DUMMY' class="ext">SM00015 </a>
            <span>(IQ)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000048-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS50096" title="PS50096 (IQ)"
       class="neutral">
        PS50096
</a>

                <span>(IQ)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS50096-span-46"
      class="match prosite-profiles SITE"
      style="left:  65.04%;
              width: 1.421%;"
      title="PS50096 1,374 - 1,403" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-46" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,374 - 1,403</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-47"
      class="match prosite-profiles SITE"
      style="left:  75.32%;
              width: 1.326%;"
      title="PS50096 1,591 - 1,618" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-47" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,591 - 1,618</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-48"
      class="match prosite-profiles SITE"
      style="left:  62.909%;
              width: 1.374%;"
      title="PS50096 1,329 - 1,357" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-48" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,329 - 1,357</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-49"
      class="match prosite-profiles SITE"
      style="left:  95.926%;
              width: 1.421%;"
      title="PS50096 2,026 - 2,055" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-49" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>2,026 - 2,055</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-50"
      class="match prosite-profiles SITE"
      style="left:  80.673%;
              width: 1.421%;"
      title="PS50096 1,704 - 1,733" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-50" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,704 - 1,733</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-51"
      class="match prosite-profiles SITE"
      style="left:  45.429%;
              width: 1.326%;"
      title="PS50096 960 - 987" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-51" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>960 - 987</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-52"
      class="match prosite-profiles SITE"
      style="left:  53.245%;
              width: 1.279%;"
      title="PS50096 1,125 - 1,151" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-52" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,125 - 1,151</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-53"
      class="match prosite-profiles SITE"
      style="left:  47.134%;
              width: 1.421%;"
      title="PS50096 996 - 1,025" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-53" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>996 - 1,025</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-54"
      class="match prosite-profiles SITE"
      style="left:  66.319%;
              width: 1.374%;"
      title="PS50096 1,401 - 1,429" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-54" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,401 - 1,429</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-55"
      class="match prosite-profiles SITE"
      style="left:  67.361%;
              width: 1.279%;"
      title="PS50096 1,423 - 1,449" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-55" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,423 - 1,449</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-56"
      class="match prosite-profiles SITE"
      style="left:  94.837%;
              width: 1.326%;"
      title="PS50096 2,003 - 2,030" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-56" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>2,003 - 2,030</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-57"
      class="match prosite-profiles SITE"
      style="left:  82.946%;
              width: 1.326%;"
      title="PS50096 1,752 - 1,779" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-57" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,752 - 1,779</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-58"
      class="match prosite-profiles SITE"
      style="left:  58.598%;
              width: 1.232%;"
      title="PS50096 1,238 - 1,263" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-58" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,238 - 1,263</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-59"
      class="match prosite-profiles SITE"
      style="left:  77.404%;
              width: 1.421%;"
      title="PS50096 1,635 - 1,664" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-59" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,635 - 1,664</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-60"
      class="match prosite-profiles SITE"
      style="left:  81.857%;
              width: 1.326%;"
      title="PS50096 1,729 - 1,756" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-60" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,729 - 1,756</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-61"
      class="match prosite-profiles SITE"
      style="left:  56.324%;
              width: 1.326%;"
      title="PS50096 1,190 - 1,217" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-61" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,190 - 1,217</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-62"
      class="match prosite-profiles SITE"
      style="left:  57.414%;
              width: 1.421%;"
      title="PS50096 1,213 - 1,242" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-62" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,213 - 1,242</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-63"
      class="match prosite-profiles SITE"
      style="left:  78.494%;
              width: 1.421%;"
      title="PS50096 1,658 - 1,687" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-63" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,658 - 1,687</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-64"
      class="match prosite-profiles SITE"
      style="left:  60.682%;
              width: 1.374%;"
      title="PS50096 1,282 - 1,310" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-64" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,282 - 1,310</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-65"
      class="match prosite-profiles SITE"
      style="left:  91.331%;
              width: 1.326%;"
      title="PS50096 1,929 - 1,956" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-65" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,929 - 1,956</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-66"
      class="match prosite-profiles SITE"
      style="left:  72.714%;
              width: 1.279%;"
      title="PS50096 1,536 - 1,562" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-66" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,536 - 1,562</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-67"
      class="match prosite-profiles SITE"
      style="left:  59.593%;
              width: 1.374%;"
      title="PS50096 1,259 - 1,287" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-67" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>1,259 - 1,287</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PS50096-span-68"
      class="match prosite-profiles SITE"
      style="left:  43.534%;
              width: 1.326%;"
      title="PS50096 920 - 947" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50096-popup-68" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles SITE caption_puce"></span>920 - 947</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50096' class="ext">PS50096 </a>
            <span>(IQ)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR000048-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00612" title="PF00612 (IQ)"
       class="neutral">
        PF00612
</a>

                <span>(IQ)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00612-span-69"
      class="match pfam SITE"
      style="left:  78.541%;
              width: 0.995%;"
      title="PF00612 1,659 - 1,679" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-69" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,659 - 1,679</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-70"
      class="match pfam SITE"
      style="left:  95.073%;
              width: 0.758%;"
      title="PF00612 2,008 - 2,023" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-70" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>2,008 - 2,023</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-71"
      class="match pfam SITE"
      style="left:  43.629%;
              width: 0.758%;"
      title="PF00612 922 - 937" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-71" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>922 - 937</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-72"
      class="match pfam SITE"
      style="left:  67.551%;
              width: 0.805%;"
      title="PF00612 1,427 - 1,443" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-72" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,427 - 1,443</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-73"
      class="match pfam SITE"
      style="left:  59.687%;
              width: 0.758%;"
      title="PF00612 1,261 - 1,276" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-73" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,261 - 1,276</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-74"
      class="match pfam SITE"
      style="left:  53.482%;
              width: 0.758%;"
      title="PF00612 1,130 - 1,145" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-74" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,130 - 1,145</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-75"
      class="match pfam SITE"
      style="left:  61.819%;
              width: 0.947%;"
      title="PF00612 1,306 - 1,325" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-75" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,306 - 1,325</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-76"
      class="match pfam SITE"
      style="left:  77.499%;
              width: 0.947%;"
      title="PF00612 1,637 - 1,656" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-76" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,637 - 1,656</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-77"
      class="match pfam SITE"
      style="left:  62.909%;
              width: 0.947%;"
      title="PF00612 1,329 - 1,348" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-77" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,329 - 1,348</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-78"
      class="match pfam SITE"
      style="left:  45.476%;
              width: 0.947%;"
      title="PF00612 961 - 980" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-78" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>961 - 980</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-79"
      class="match pfam SITE"
      style="left:  57.508%;
              width: 0.9%;"
      title="PF00612 1,215 - 1,233" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-79" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,215 - 1,233</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-80"
      class="match pfam SITE"
      style="left:  95.973%;
              width: 0.9%;"
      title="PF00612 2,027 - 2,045" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-80" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>2,027 - 2,045</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-81"
      class="match pfam SITE"
      style="left:  56.371%;
              width: 0.995%;"
      title="PF00612 1,191 - 1,211" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-81" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,191 - 1,211</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-82"
      class="match pfam SITE"
      style="left:  81.857%;
              width: 0.805%;"
      title="PF00612 1,729 - 1,745" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-82" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,729 - 1,745</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-83"
      class="match pfam SITE"
      style="left:  80.815%;
              width: 0.758%;"
      title="PF00612 1,707 - 1,722" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-83" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,707 - 1,722</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-84"
      class="match pfam SITE"
      style="left:  47.229%;
              width: 0.9%;"
      title="PF00612 998 - 1,016" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-84" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>998 - 1,016</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-85"
      class="match pfam SITE"
      style="left:  91.473%;
              width: 0.853%;"
      title="PF00612 1,932 - 1,949" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-85" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,932 - 1,949</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-86"
      class="match pfam SITE"
      style="left:  69.588%;
              width: 0.9%;"
      title="PF00612 1,470 - 1,488" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-86" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,470 - 1,488</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-87"
      class="match pfam SITE"
      style="left:  58.598%;
              width: 0.9%;"
      title="PF00612 1,238 - 1,256" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-87" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,238 - 1,256</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-88"
      class="match pfam SITE"
      style="left:  64.14%;
              width: 0.805%;"
      title="PF00612 1,355 - 1,371" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-88" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,355 - 1,371</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-89"
      class="match pfam SITE"
      style="left:  72.809%;
              width: 0.758%;"
      title="PF00612 1,538 - 1,553" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-89" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,538 - 1,553</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


<span id="PF00612-span-90"
      class="match pfam SITE"
      style="left:  71.767%;
              width: 0.853%;"
      title="PF00612 1,516 - 1,533" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00612-popup-90" style="display: none;">

        <div class="popup_topl"><span class="pfam SITE caption_puce"></span>1,516 - 1,533</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00612' class="ext">PF00612 </a>
            <span>(IQ)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.474%;" title="200"></span>
            <span class="grade" style="left:18.948%;" title="400"></span>
            <span class="grade" style="left:28.423%;" title="600"></span>
            <span class="grade" style="left:37.897%;" title="800"></span>
            <span class="grade" style="left:47.371%;" title="1000"></span>
            <span class="grade" style="left:56.845%;" title="1200"></span>
            <span class="grade" style="left:66.319%;" title="1400"></span>
            <span class="grade" style="left:75.793%;" title="1600"></span>
            <span class="grade" style="left:85.268%;" title="1800"></span>
            <span class="grade" style="left:94.742%;" title="2000"></span>
            <span class="grade" style="left:100%;" title="2111"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>



<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515"
                   class="ext">GO:0005515</a> protein binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 6);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
