<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8" content="text/html">
    <title>
    comp1736_c0_seq0|m.7625 - InterPro
    </title>
    <meta name="description" content="InterProScan result page: Protein matches and sequence features">
    <meta name="author" content="InterPro Team, European Bioinformatics Institute">


    <link href="resources/css/type_colours.css" rel="stylesheet" type="text/css"/>
    <link class="database" href="resources/css/database.css" rel="stylesheet" type="text/css"/>
    <link href="resources/javascript/qtip2/jquery.qtip.css" rel="stylesheet" type="text/css"/>
    <link href="resources/css/protein.css" rel="stylesheet" type="text/css"/>
    <script src="resources/javascript/jquery/jquery-1.7.1.min.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery-ui-1.8.17.custom.min.js" type="text/javascript"></script>
    <script src="resources/javascript/qtip2/jquery.qtip.min.js" type="text/javascript"></script>
    <script src="resources/javascript/common.js" type="text/javascript"></script>
    <script src="resources/javascript/protein.js" type="text/javascript"></script>
    <script src="resources/javascript/protein-popups.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.cookie.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.jscroll.min.js" type="text/javascript"></script>

</head>
<body>
<div class="container_24">
<div class="grid_24 clearfix" id="content" >

    <div class="grid_5 alpha">
<div id="menu" style="display: none;">
    <div class="menu-filter">

        <h1>Filter view on</h1>

        <div class="menu-filter-type"><h1>Entry type</h1>

            <fieldset>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-2" value="Family-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-2" class="family">Family</label>
                </div>
                <div >
                    <input type="checkbox" class="type" id="check-3" value="Domain-row"
                           checked="checked" />
                    <label for="check-3" class="domains">Domains</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-4" value="Repeat-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-4" class="repeats">Repeats</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-5"
                           value="SITE-row Conserved_site-row PTM-row Binding_site-row Active_site-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-5" class="site">Site</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-status"><h1>Status</h1>

            <fieldset>
                <div class="disabled">
                    <input type="checkbox" name="type" id="check-6" value="unintegrated"
                           checked="checked" disabled="disabled"/>
                    <label for="check-6" class="uni">Unintegrated</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-colour">
            <div class="filter-colour-head">
                <div class="filter-colour-title-l">
                    Colour by
                </div>

            </div>
            <fieldset>
                <input type="radio" name="blobColour" id="domainColour" value="domainColour" checked="checked"/>
                <label for="domainColour">domain relationship</label> <br/>

                <input type="radio" name="blobColour" id="databaseColour" value="databaseColour"/>
                <label for="databaseColour">source database</label>
            </fieldset>
        </div>
    </div>

</div>
<script type="text/javascript">
    $(document).ready(function() {
// Read colour preference from cookie (requires http://plugins.jquery.com/project/Cookie)

            // Retrieve existing cookies and set checkbox states accordingly
            var checkBoxIDs = ["#check-2", "#check-3", "#check-4", "#check-5", "#check-6"];
            for (i = 0; i < checkBoxIDs.length; i++) {
                var checkBoxId = checkBoxIDs[i];
                var cookieVal = $.cookie(checkBoxId);
                if (cookieVal != null) {
                    $(checkBoxId).prop("checked", cookieVal == "true");
                }
            }

            // and the radio button group
            var radioCookieVal = $.cookie("colour-by-domain");
            if (radioCookieVal != null) {
                if (radioCookieVal == "true") {
                    $('#domainColour').prop("checked", true);
                }
                else {
                    $('#databaseColour').prop("checked", true);
                }
            }

            // Keep the filter menu in view
//            $("#menu").jScroll();
            // CSS switching
            configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour'); // initialise

            $('input[name="blobColour"]').change(function() {
                configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour');
            });

            // Change event for type checkboxes (Family, Domain etc.)
            $(".type").change(function() {
                displayType(this);
            });

            // Initialise types
            $(".type").each(function() {
                displayType(this);
            });

            // Change event for un-integrated sig matches checkbox
            $("#check-6").change(function() {
                displayUnintegrated(this);
            });

            // Initialise un-integrated.
            // TODO Work out how to do this without each! Only one thing with an ID of "check-6"!
            $("#check-6").each(function() {
                displayUnintegrated(this);
            });

        // Make the menu visible - Javascript is enabled.
        $("#menu").css('display', 'block');
    });
</script>
    </div>

    <div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp1736_c0_seq0|m.7625 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">265 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




        <h3>Protein family membership</h3>
        <p>None predicted.</p>



    <h3>Domains and repeats</h3>


    <div class="prot_sum">
    <div class="bot-row">
        <div class="bot-row-line-top"></div>
        <ol class="signatures">



    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-1"
      class="match c35 Domain"
      style="left:  23.396%;
              width: 14.34%;"
      title="Domain 63 - 100" >
</span>
        <div id="supermatch-popup-1" style="display: none;">
        <div class="popup_topl">
            <span class="c35 Domain caption_puce"></span>63 - 100
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR001878">Zinc finger, CCHC-type</a><span> (IPR001878)</span></li></ul></div>
    </div>




<span id="supermatch-span-2"
      class="match c35 Domain"
      style="left:  37.736%;
              width: 58.491%;"
      title="Domain 101 - 255" >
</span>
        <div id="supermatch-popup-2" style="display: none;">
        <div class="popup_topl">
            <span class="c35 Domain caption_puce"></span>101 - 255
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR001878">Zinc finger, CCHC-type</a><span> (IPR001878)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:18.868%;" title="50"></span>
                    <span class="grade" style="left:37.736%;" title="100"></span>
                    <span class="grade" style="left:56.604%;" title="150"></span>
                    <span class="grade" style="left:75.472%;" title="200"></span>
                    <span class="grade" style="left:100%;" title="265"></span>

            </div>
        </div>
    </li>

        </ol>
        <div class="bot-row-line-bot"></div>
    </div>

    <div class="prot_scale">
        <div class="bot-row">

            <div class="bot-row-line">
                <div style="position:relative;">
                            <span class="scale_bar" style="left:0%;"
                                  title="1"></span>
                                <span class="scale_numb"
                                      style="left:1%;">1</span>
                            <span class="scale_bar" style="left:18.868%;"
                                  title="50"></span>
                                <span class="scale_numb"
                                      style="left:18.868%;">50</span>
                            <span class="scale_bar" style="left:37.736%;"
                                  title="100"></span>
                                <span class="scale_numb"
                                      style="left:37.736%;">100</span>
                            <span class="scale_bar" style="left:56.604%;"
                                  title="150"></span>
                                <span class="scale_numb"
                                      style="left:56.604%;">150</span>
                            <span class="scale_bar" style="left:75.472%;"
                                  title="200"></span>
                                <span class="scale_numb"
                                      style="left:75.472%;">200</span>
                            <span class="scale_bar" style="left:100%;"
                                  title="265"></span>
                                <span class="scale_numb"
                                      style="left:100%;">265</span>
                </div>
            </div>

        </div>
    </div>

    </div>



    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR001878">IPR001878</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR001878"
                                                         class="neutral">Zinc finger, CCHC-type</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR001878-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF57756" title="SSF57756 (Retrovirus zinc finger-like domains)"
       class="neutral">
        SSF57756
</a>

            <span>(Retroviru...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF57756-span-1"
      class="match superfamily c35 Domain"
      style="left:  37.736%;
              width: 13.962%;"
      title="SSF57756 101 - 137" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF57756-popup-1" style="display: none;">

        <div class="popup_topl"><span class="superfamily c35 Domain caption_puce"></span>101 - 137</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF57756' class="ext">SSF57756 </a>
            <span>(Retrovirus zinc finger-like domains)</span>
        </div>
    </div>


<span id="SSF57756-span-2"
      class="match superfamily c35 Domain"
      style="left:  68.679%;
              width: 7.547%;"
      title="SSF57756 183 - 202" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF57756-popup-2" style="display: none;">

        <div class="popup_topl"><span class="superfamily c35 Domain caption_puce"></span>183 - 202</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF57756' class="ext">SSF57756 </a>
            <span>(Retrovirus zinc finger-like domains)</span>
        </div>
    </div>


<span id="SSF57756-span-3"
      class="match superfamily c35 Domain"
      style="left:  79.623%;
              width: 16.604%;"
      title="SSF57756 212 - 255" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF57756-popup-3" style="display: none;">

        <div class="popup_topl"><span class="superfamily c35 Domain caption_puce"></span>212 - 255</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF57756' class="ext">SSF57756 </a>
            <span>(Retrovirus zinc finger-like domains)</span>
        </div>
    </div>


<span id="SSF57756-span-4"
      class="match superfamily c35 Domain"
      style="left:  53.585%;
              width: 15.472%;"
      title="SSF57756 143 - 183" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF57756-popup-4" style="display: none;">

        <div class="popup_topl"><span class="superfamily c35 Domain caption_puce"></span>143 - 183</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF57756' class="ext">SSF57756 </a>
            <span>(Retrovirus zinc finger-like domains)</span>
        </div>
    </div>


<span id="SSF57756-span-5"
      class="match superfamily c35 Domain"
      style="left:  23.396%;
              width: 14.34%;"
      title="SSF57756 63 - 100" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF57756-popup-5" style="display: none;">

        <div class="popup_topl"><span class="superfamily c35 Domain caption_puce"></span>63 - 100</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF57756' class="ext">SSF57756 </a>
            <span>(Retrovirus zinc finger-like domains)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:18.868%;" title="50"></span>
            <span class="grade" style="left:37.736%;" title="100"></span>
            <span class="grade" style="left:56.604%;" title="150"></span>
            <span class="grade" style="left:75.472%;" title="200"></span>
            <span class="grade" style="left:100%;" title="265"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001878-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY" title="SM00343 (ZnF_C2HC)"
       class="neutral">
        SM00343
</a>

                <span>(ZnF_C2HC)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00343-span-6"
      class="match smart c35 Domain"
      style="left:  69.434%;
              width: 6.415%;"
      title="SM00343 185 - 201" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-6" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>185 - 201</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-7"
      class="match smart c35 Domain"
      style="left:  45.283%;
              width: 6.415%;"
      title="SM00343 121 - 137" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-7" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>121 - 137</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-8"
      class="match smart c35 Domain"
      style="left:  30.943%;
              width: 6.415%;"
      title="SM00343 83 - 99" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-8" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>83 - 99</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-9"
      class="match smart c35 Domain"
      style="left:  55.094%;
              width: 6.415%;"
      title="SM00343 147 - 163" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-9" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>147 - 163</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-10"
      class="match smart c35 Domain"
      style="left:  62.264%;
              width: 6.415%;"
      title="SM00343 166 - 182" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-10" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>166 - 182</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-11"
      class="match smart c35 Domain"
      style="left:  82.264%;
              width: 6.415%;"
      title="SM00343 219 - 235" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-11" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>219 - 235</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-12"
      class="match smart c35 Domain"
      style="left:  38.113%;
              width: 6.415%;"
      title="SM00343 102 - 118" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-12" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>102 - 118</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-13"
      class="match smart c35 Domain"
      style="left:  89.434%;
              width: 6.415%;"
      title="SM00343 238 - 254" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-13" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>238 - 254</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


<span id="SM00343-span-14"
      class="match smart c35 Domain"
      style="left:  23.774%;
              width: 6.415%;"
      title="SM00343 64 - 80" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00343-popup-14" style="display: none;">

        <div class="popup_topl"><span class="smart c35 Domain caption_puce"></span>64 - 80</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00343&amp;BLAST=DUMMY' class="ext">SM00343 </a>
            <span>(ZnF_C2HC)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:18.868%;" title="50"></span>
            <span class="grade" style="left:37.736%;" title="100"></span>
            <span class="grade" style="left:56.604%;" title="150"></span>
            <span class="grade" style="left:75.472%;" title="200"></span>
            <span class="grade" style="left:100%;" title="265"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001878-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS50158" title="PS50158 (ZF_CCHC)"
       class="neutral">
        PS50158
</a>

                <span>(ZF_CCHC)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS50158-span-15"
      class="match prosite-profiles c35 Domain"
      style="left:  31.321%;
              width: 5.66%;"
      title="PS50158 84 - 98" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-15" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>84 - 98</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-16"
      class="match prosite-profiles c35 Domain"
      style="left:  55.472%;
              width: 6.038%;"
      title="PS50158 148 - 163" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-16" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>148 - 163</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-17"
      class="match prosite-profiles c35 Domain"
      style="left:  62.642%;
              width: 5.283%;"
      title="PS50158 167 - 180" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-17" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>167 - 180</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-18"
      class="match prosite-profiles c35 Domain"
      style="left:  45.66%;
              width: 5.283%;"
      title="PS50158 122 - 135" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-18" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>122 - 135</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-19"
      class="match prosite-profiles c35 Domain"
      style="left:  89.811%;
              width: 6.038%;"
      title="PS50158 239 - 254" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-19" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>239 - 254</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-20"
      class="match prosite-profiles c35 Domain"
      style="left:  69.811%;
              width: 6.038%;"
      title="PS50158 186 - 201" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-20" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>186 - 201</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-21"
      class="match prosite-profiles c35 Domain"
      style="left:  24.151%;
              width: 6.038%;"
      title="PS50158 65 - 80" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-21" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>65 - 80</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-22"
      class="match prosite-profiles c35 Domain"
      style="left:  82.642%;
              width: 5.283%;"
      title="PS50158 220 - 233" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-22" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>220 - 233</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


<span id="PS50158-span-23"
      class="match prosite-profiles c35 Domain"
      style="left:  38.491%;
              width: 5.283%;"
      title="PS50158 103 - 116" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50158-popup-23" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c35 Domain caption_puce"></span>103 - 116</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50158' class="ext">PS50158 </a>
            <span>(ZF_CCHC)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:18.868%;" title="50"></span>
            <span class="grade" style="left:37.736%;" title="100"></span>
            <span class="grade" style="left:56.604%;" title="150"></span>
            <span class="grade" style="left:75.472%;" title="200"></span>
            <span class="grade" style="left:100%;" title="265"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001878-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/4.10.60.10" title="G3DSA:4.10.60.10"
           class="neutral">
        G3DSA:4.10.60.10
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA4106010-span-24"
      class="match gene3d c35 Domain"
      style="left:  72.83%;
              width: 12.453%;"
      title="G3DSA:4.10.60.10 194 - 226" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-24" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>194 - 226</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


<span id="G3DSA4106010-span-25"
      class="match gene3d c35 Domain"
      style="left:  48.679%;
              width: 10.189%;"
      title="G3DSA:4.10.60.10 130 - 156" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-25" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>130 - 156</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


<span id="G3DSA4106010-span-26"
      class="match gene3d c35 Domain"
      style="left:  58.868%;
              width: 6.792%;"
      title="G3DSA:4.10.60.10 157 - 174" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-26" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>157 - 174</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


<span id="G3DSA4106010-span-27"
      class="match gene3d c35 Domain"
      style="left:  24.151%;
              width: 13.208%;"
      title="G3DSA:4.10.60.10 65 - 99" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-27" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>65 - 99</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


<span id="G3DSA4106010-span-28"
      class="match gene3d c35 Domain"
      style="left:  38.491%;
              width: 10.189%;"
      title="G3DSA:4.10.60.10 103 - 129" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-28" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>103 - 129</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


<span id="G3DSA4106010-span-29"
      class="match gene3d c35 Domain"
      style="left:  85.283%;
              width: 10.566%;"
      title="G3DSA:4.10.60.10 227 - 254" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-29" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>227 - 254</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


<span id="G3DSA4106010-span-30"
      class="match gene3d c35 Domain"
      style="left:  65.66%;
              width: 7.17%;"
      title="G3DSA:4.10.60.10 175 - 193" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA4106010-popup-30" style="display: none;">

        <div class="popup_topl"><span class="gene3d c35 Domain caption_puce"></span>175 - 193</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/4.10.60.10' class="ext">G3DSA:4.10.60.10 </a>
            <span>(G3DSA:4.10.60.10)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:18.868%;" title="50"></span>
            <span class="grade" style="left:37.736%;" title="100"></span>
            <span class="grade" style="left:56.604%;" title="150"></span>
            <span class="grade" style="left:75.472%;" title="200"></span>
            <span class="grade" style="left:100%;" title="265"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001878-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00098" title="PF00098 (zf-CCHC)"
       class="neutral">
        PF00098
</a>

                <span>(zf-CCHC)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00098-span-31"
      class="match pfam c35 Domain"
      style="left:  89.434%;
              width: 6.415%;"
      title="PF00098 238 - 254" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-31" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>238 - 254</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-32"
      class="match pfam c35 Domain"
      style="left:  30.943%;
              width: 6.038%;"
      title="PF00098 83 - 98" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-32" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>83 - 98</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-33"
      class="match pfam c35 Domain"
      style="left:  24.151%;
              width: 6.038%;"
      title="PF00098 65 - 80" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-33" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>65 - 80</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-34"
      class="match pfam c35 Domain"
      style="left:  55.094%;
              width: 6.415%;"
      title="PF00098 147 - 163" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-34" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>147 - 163</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-35"
      class="match pfam c35 Domain"
      style="left:  38.113%;
              width: 5.66%;"
      title="PF00098 102 - 116" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-35" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>102 - 116</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-36"
      class="match pfam c35 Domain"
      style="left:  69.434%;
              width: 6.038%;"
      title="PF00098 185 - 200" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-36" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>185 - 200</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-37"
      class="match pfam c35 Domain"
      style="left:  62.264%;
              width: 5.66%;"
      title="PF00098 166 - 180" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-37" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>166 - 180</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


<span id="PF00098-span-38"
      class="match pfam c35 Domain"
      style="left:  82.264%;
              width: 5.66%;"
      title="PF00098 219 - 233" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00098-popup-38" style="display: none;">

        <div class="popup_topl"><span class="pfam c35 Domain caption_puce"></span>219 - 233</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00098' class="ext">PF00098 </a>
            <span>(zf-CCHC)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:18.868%;" title="50"></span>
            <span class="grade" style="left:37.736%;" title="100"></span>
            <span class="grade" style="left:56.604%;" title="150"></span>
            <span class="grade" style="left:75.472%;" title="200"></span>
            <span class="grade" style="left:100%;" title="265"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>



<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003676"
                   class="ext">GO:0003676</a> nucleic acid binding
                <br/>
            
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008270"
                   class="ext">GO:0008270</a> zinc ion binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 2);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
</body>
</html>
