<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8" content="text/html">
    <title>
    comp33171_c0_seq0|m.48178 - InterPro
    </title>
    <meta name="description" content="InterProScan result page: Protein matches and sequence features">
    <meta name="author" content="InterPro Team, European Bioinformatics Institute">


    <link href="resources/css/type_colours.css" rel="stylesheet" type="text/css"/>
    <link class="database" href="resources/css/database.css" rel="stylesheet" type="text/css"/>
    <link href="resources/javascript/qtip2/jquery.qtip.css" rel="stylesheet" type="text/css"/>
    <link href="resources/css/protein.css" rel="stylesheet" type="text/css"/>
    <script src="resources/javascript/jquery/jquery-1.7.1.min.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery-ui-1.8.17.custom.min.js" type="text/javascript"></script>
    <script src="resources/javascript/qtip2/jquery.qtip.min.js" type="text/javascript"></script>
    <script src="resources/javascript/common.js" type="text/javascript"></script>
    <script src="resources/javascript/protein.js" type="text/javascript"></script>
    <script src="resources/javascript/protein-popups.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.cookie.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.jscroll.min.js" type="text/javascript"></script>

</head>
<body>
<div class="container_24">
<div class="grid_24 clearfix" id="content" >

    <div class="grid_5 alpha">
<div id="menu" style="display: none;">
    <div class="menu-filter">

        <h1>Filter view on</h1>

        <div class="menu-filter-type"><h1>Entry type</h1>

            <fieldset>
                <div >
                    <input type="checkbox" class="type" id="check-2" value="Family-row"
                           checked="checked" />
                    <label for="check-2" class="family">Family</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-3" value="Domain-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-3" class="domains">Domains</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-4" value="Repeat-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-4" class="repeats">Repeats</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-5"
                           value="SITE-row Conserved_site-row PTM-row Binding_site-row Active_site-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-5" class="site">Site</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-status"><h1>Status</h1>

            <fieldset>
                <div >
                    <input type="checkbox" name="type" id="check-6" value="unintegrated"
                           checked="checked" />
                    <label for="check-6" class="uni">Unintegrated</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-colour">
            <div class="filter-colour-head">
                <div class="filter-colour-title-l">
                    Colour by
                </div>

            </div>
            <fieldset>
                <input type="radio" name="blobColour" id="domainColour" value="domainColour" checked="checked"/>
                <label for="domainColour">domain relationship</label> <br/>

                <input type="radio" name="blobColour" id="databaseColour" value="databaseColour"/>
                <label for="databaseColour">source database</label>
            </fieldset>
        </div>
    </div>

</div>
<script type="text/javascript">
    $(document).ready(function() {
// Read colour preference from cookie (requires http://plugins.jquery.com/project/Cookie)

            // Retrieve existing cookies and set checkbox states accordingly
            var checkBoxIDs = ["#check-2", "#check-3", "#check-4", "#check-5", "#check-6"];
            for (i = 0; i < checkBoxIDs.length; i++) {
                var checkBoxId = checkBoxIDs[i];
                var cookieVal = $.cookie(checkBoxId);
                if (cookieVal != null) {
                    $(checkBoxId).prop("checked", cookieVal == "true");
                }
            }

            // and the radio button group
            var radioCookieVal = $.cookie("colour-by-domain");
            if (radioCookieVal != null) {
                if (radioCookieVal == "true") {
                    $('#domainColour').prop("checked", true);
                }
                else {
                    $('#databaseColour').prop("checked", true);
                }
            }

            // Keep the filter menu in view
//            $("#menu").jScroll();
            // CSS switching
            configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour'); // initialise

            $('input[name="blobColour"]').change(function() {
                configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour');
            });

            // Change event for type checkboxes (Family, Domain etc.)
            $(".type").change(function() {
                displayType(this);
            });

            // Initialise types
            $(".type").each(function() {
                displayType(this);
            });

            // Change event for un-integrated sig matches checkbox
            $("#check-6").change(function() {
                displayUnintegrated(this);
            });

            // Initialise un-integrated.
            // TODO Work out how to do this without each! Only one thing with an ID of "check-6"!
            $("#check-6").each(function() {
                displayUnintegrated(this);
            });

        // Make the menu visible - Javascript is enabled.
        $("#menu").css('display', 'block');
    });
</script>
    </div>

    <div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp33171_c0_seq0|m.48178 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">132 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




          <h3>Protein family membership</h3>
          <div class="Family rel_tree"><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR005455">Profilin</a><span> (IPR005455)</span></li></ul></div>



    <h3>Domains and repeats</h3>


    <div class="prot_sum" style="background:none;">
    <div class="bot-row">
        None predicted.
    </div>
    </div>


    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Family-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_family_small.png"
                                                         alt="Family" title="Family"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR005455">IPR005455</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR005455"
                                                         class="neutral">Profilin</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR005455-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00392 (PROFILIN)"
       class="neutral">
        PR00392
</a>

                <span>(PROFILIN)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00392-span-1"
      class="match prints Family"
      style="left:  30.303%;
              width: 15.909%;"
      title="PR00392 41 - 61" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00392-popup-1" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>41 - 61</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00392 </a>
            <span>(PROFILIN)</span>
        </div>
    </div>


<span id="PR00392-span-2"
      class="match prints Family"
      style="left:  1.515%;
              width: 7.576%;"
      title="PR00392 3 - 12" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00392-popup-2" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>3 - 12</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00392 </a>
            <span>(PROFILIN)</span>
        </div>
    </div>


<span id="PR00392-span-3"
      class="match prints Family"
      style="left:  48.485%;
              width: 11.364%;"
      title="PR00392 65 - 79" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00392-popup-3" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>65 - 79</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00392 </a>
            <span>(PROFILIN)</span>
        </div>
    </div>


<span id="PR00392-span-4"
      class="match prints Family"
      style="left:  15.909%;
              width: 7.576%;"
      title="PR00392 22 - 31" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00392-popup-4" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>22 - 31</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00392 </a>
            <span>(PROFILIN)</span>
        </div>
    </div>


<span id="PR00392-span-5"
      class="match prints Family"
      style="left:  85.606%;
              width: 13.636%;"
      title="PR00392 114 - 131" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00392-popup-5" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>114 - 131</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00392 </a>
            <span>(PROFILIN)</span>
        </div>
    </div>


<span id="PR00392-span-6"
      class="match prints Family"
      style="left:  75.758%;
              width: 10.606%;"
      title="PR00392 101 - 114" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00392-popup-6" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>101 - 114</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00392&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00392 </a>
            <span>(PROFILIN)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.152%;" title="20"></span>
            <span class="grade" style="left:30.303%;" title="40"></span>
            <span class="grade" style="left:45.455%;" title="60"></span>
            <span class="grade" style="left:60.606%;" title="80"></span>
            <span class="grade" style="left:75.758%;" title="100"></span>
            <span class="grade" style="left:90.909%;" title="120"></span>
            <span class="grade" style="left:100%;" title="132"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR005455-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00392&amp;BLAST=DUMMY" title="SM00392 (PROF)"
       class="neutral">
        SM00392
</a>

                <span>(PROF)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SM00392-span-7"
      class="match smart Family"
      style="left:  0%;
              width: 100%;"
      title="SM00392 1 - 132" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SM00392-popup-7" style="display: none;">

        <div class="popup_topl"><span class="smart Family caption_puce"></span>1 - 132</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SMART</b> <abbr class="icon icon-generic" data-icon="i" title="SMART (a Simple Modular Architecture Research Tool) uses hidden Markov models (HMMs) and allows the identification of genetically mobile domains and the analysis of domain architectures. These domains are extensively annotated with respect to phyletic distributions, functional class, tertiary structures and functionally important residues."></abbr> <br/>
            <a href='http://smart.embl-heidelberg.de/smart/do_annotation.pl?ACC=SM00392&amp;BLAST=DUMMY' class="ext">SM00392 </a>
            <span>(PROF)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.152%;" title="20"></span>
            <span class="grade" style="left:30.303%;" title="40"></span>
            <span class="grade" style="left:45.455%;" title="60"></span>
            <span class="grade" style="left:60.606%;" title="80"></span>
            <span class="grade" style="left:75.758%;" title="100"></span>
            <span class="grade" style="left:90.909%;" title="120"></span>
            <span class="grade" style="left:100%;" title="132"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR005455-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF55770" title="SSF55770 (Profilin (actin-binding protein))"
       class="neutral">
        SSF55770
</a>

            <span>(Profilin ...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF55770-span-8"
      class="match superfamily Family"
      style="left:  0.758%;
              width: 99.242%;"
      title="SSF55770 2 - 132" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF55770-popup-8" style="display: none;">

        <div class="popup_topl"><span class="superfamily Family caption_puce"></span>2 - 132</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF55770' class="ext">SSF55770 </a>
            <span>(Profilin (actin-binding protein))</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.152%;" title="20"></span>
            <span class="grade" style="left:30.303%;" title="40"></span>
            <span class="grade" style="left:45.455%;" title="60"></span>
            <span class="grade" style="left:60.606%;" title="80"></span>
            <span class="grade" style="left:75.758%;" title="100"></span>
            <span class="grade" style="left:90.909%;" title="120"></span>
            <span class="grade" style="left:100%;" title="132"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR005455-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00235" title="PF00235 (Profilin)"
       class="neutral">
        PF00235
</a>

                <span>(Profilin)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00235-span-9"
      class="match pfam Family"
      style="left:  0.758%;
              width: 95.455%;"
      title="PF00235 2 - 127" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00235-popup-9" style="display: none;">

        <div class="popup_topl"><span class="pfam Family caption_puce"></span>2 - 127</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00235' class="ext">PF00235 </a>
            <span>(Profilin)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.152%;" title="20"></span>
            <span class="grade" style="left:30.303%;" title="40"></span>
            <span class="grade" style="left:45.455%;" title="60"></span>
            <span class="grade" style="left:60.606%;" title="80"></span>
            <span class="grade" style="left:75.758%;" title="100"></span>
            <span class="grade" style="left:90.909%;" title="120"></span>
            <span class="grade" style="left:100%;" title="132"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR005455-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR01640 (PROFILINPLNT)"
       class="neutral">
        PR01640
</a>

                <span>(PROFILINPLNT)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR01640-span-10"
      class="match prints Family"
      style="left:  0%;
              width: 10.606%;"
      title="PR01640 1 - 14" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01640-popup-10" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>1 - 14</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01640 </a>
            <span>(PROFILINPLNT)</span>
        </div>
    </div>


<span id="PR01640-span-11"
      class="match prints Family"
      style="left:  63.636%;
              width: 12.121%;"
      title="PR01640 85 - 100" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01640-popup-11" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>85 - 100</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01640 </a>
            <span>(PROFILINPLNT)</span>
        </div>
    </div>


<span id="PR01640-span-12"
      class="match prints Family"
      style="left:  20.455%;
              width: 10.606%;"
      title="PR01640 28 - 41" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01640-popup-12" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>28 - 41</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01640 </a>
            <span>(PROFILINPLNT)</span>
        </div>
    </div>


<span id="PR01640-span-13"
      class="match prints Family"
      style="left:  88.636%;
              width: 10.606%;"
      title="PR01640 118 - 131" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01640-popup-13" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>118 - 131</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01640 </a>
            <span>(PROFILINPLNT)</span>
        </div>
    </div>


<span id="PR01640-span-14"
      class="match prints Family"
      style="left:  78.788%;
              width: 10.606%;"
      title="PR01640 105 - 118" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01640-popup-14" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>105 - 118</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01640 </a>
            <span>(PROFILINPLNT)</span>
        </div>
    </div>


<span id="PR01640-span-15"
      class="match prints Family"
      style="left:  53.788%;
              width: 7.576%;"
      title="PR01640 72 - 81" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR01640-popup-15" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>72 - 81</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR01640&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR01640 </a>
            <span>(PROFILINPLNT)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.152%;" title="20"></span>
            <span class="grade" style="left:30.303%;" title="40"></span>
            <span class="grade" style="left:45.455%;" title="60"></span>
            <span class="grade" style="left:60.606%;" title="80"></span>
            <span class="grade" style="left:75.758%;" title="100"></span>
            <span class="grade" style="left:90.909%;" title="120"></span>
            <span class="grade" style="left:100%;" title="132"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>


            <div class="prot_entries" id="uni_sign">
                <div class="top-row">
                    <div class="top-row-id"><img src="resources/images/ico_type_uni_small.png"
                                                 alt="Unintegrated signatures" title="Unintegrated signatures"/> no IPR
                    </div>
                    <div class="top-row-name">Unintegrated signatures</div>
                </div>
                <div class="bot-row">
                    <div class="bot-row-line-top"></div>
                    <ol class="signatures">
                            <li class="signature">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/3.30.450.30" title="G3DSA:3.30.450.30"
           class="neutral">
    G3DSA:3.30.45...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA33045030-span-16"
      class="match gene3d uni"
      style="left:  0.758%;
              width: 98.485%;"
      title="G3DSA:3.30.450.30 2 - 131" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA33045030-popup-16" style="display: none;">

        <div class="popup_topl"><span class="gene3d uni caption_puce"></span>2 - 131</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/3.30.450.30' class="ext">G3DSA:3.30.450.30 </a>
            <span>(G3DSA:3.30.450.30)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.152%;" title="20"></span>
            <span class="grade" style="left:30.303%;" title="40"></span>
            <span class="grade" style="left:45.455%;" title="60"></span>
            <span class="grade" style="left:60.606%;" title="80"></span>
            <span class="grade" style="left:75.758%;" title="100"></span>
            <span class="grade" style="left:90.909%;" title="120"></span>
            <span class="grade" style="left:100%;" title="132"></span>

    </div>

</div>
                            </li>
                    </ol>
                    <div class="bot-row-line-bot"></div>
                </div>
            </div>


<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 0);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
</body>
</html>
