<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8" content="text/html">
    <title>
    comp3725_c0_seq0|m.16571 - InterPro
    </title>
    <meta name="description" content="InterProScan result page: Protein matches and sequence features">
    <meta name="author" content="InterPro Team, European Bioinformatics Institute">


    <link href="resources/css/type_colours.css" rel="stylesheet" type="text/css"/>
    <link class="database" href="resources/css/database.css" rel="stylesheet" type="text/css"/>
    <link href="resources/javascript/qtip2/jquery.qtip.css" rel="stylesheet" type="text/css"/>
    <link href="resources/css/protein.css" rel="stylesheet" type="text/css"/>
    <script src="resources/javascript/jquery/jquery-1.7.1.min.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery-ui-1.8.17.custom.min.js" type="text/javascript"></script>
    <script src="resources/javascript/qtip2/jquery.qtip.min.js" type="text/javascript"></script>
    <script src="resources/javascript/common.js" type="text/javascript"></script>
    <script src="resources/javascript/protein.js" type="text/javascript"></script>
    <script src="resources/javascript/protein-popups.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.cookie.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.jscroll.min.js" type="text/javascript"></script>

</head>
<body>
<div class="container_24">
<div class="grid_24 clearfix" id="content" >


    <div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp3725_c0_seq0|m.16571 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">345 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




        <h3>Protein family membership</h3>
        <p>None predicted.</p>



    <h3>Domains and repeats</h3>


    <div class="prot_sum" style="background:none;">
    <div class="bot-row">
        None predicted.
    </div>
    </div>


    <h3>Detailed signature matches</h3>




            <div class="prot_entries" id="uni_sign">
                <div class="top-row">
                    <div class="top-row-id"><img src="resources/images/ico_type_uni_small.png"
                                                 alt="Unintegrated signatures" title="Unintegrated signatures"/> no IPR
                    </div>
                    <div class="top-row-name">Unintegrated signatures</div>
                </div>
                <div class="bot-row">
                    <div class="bot-row-line-top"></div>
                    <ol class="signatures">
                            <li class="signature">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/1.20.5.110" title="G3DSA:1.20.5.110"
           class="neutral">
        G3DSA:1.20.5.110
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA1205110-span-1"
      class="match gene3d uni"
      style="left:  72.174%;
              width: 18.841%;"
      title="G3DSA:1.20.5.110 250 - 314" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA1205110-popup-1" style="display: none;">

        <div class="popup_topl"><span class="gene3d uni caption_puce"></span>250 - 314</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.20.5.110' class="ext">G3DSA:1.20.5.110 </a>
            <span>(G3DSA:1.20.5.110)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:14.493%;" title="50"></span>
            <span class="grade" style="left:28.986%;" title="100"></span>
            <span class="grade" style="left:43.478%;" title="150"></span>
            <span class="grade" style="left:57.971%;" title="200"></span>
            <span class="grade" style="left:72.464%;" title="250"></span>
            <span class="grade" style="left:86.957%;" title="300"></span>
            <span class="grade" style="left:100%;" title="345"></span>

    </div>

</div>
                            </li>
                            <li class="signature">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF12352" title="PF12352 (V-SNARE_C)"
       class="neutral">
        PF12352
</a>

                <span>(V-SNARE_C)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF12352-span-2"
      class="match pfam uni"
      style="left:  77.101%;
              width: 14.783%;"
      title="PF12352 267 - 317" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF12352-popup-2" style="display: none;">

        <div class="popup_topl"><span class="pfam uni caption_puce"></span>267 - 317</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF12352' class="ext">PF12352 </a>
            <span>(V-SNARE_C)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:14.493%;" title="50"></span>
            <span class="grade" style="left:28.986%;" title="100"></span>
            <span class="grade" style="left:43.478%;" title="150"></span>
            <span class="grade" style="left:57.971%;" title="200"></span>
            <span class="grade" style="left:72.464%;" title="250"></span>
            <span class="grade" style="left:86.957%;" title="300"></span>
            <span class="grade" style="left:100%;" title="345"></span>

    </div>

</div>
                            </li>
                            <li class="signature">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF58038" title="SSF58038 (SNARE fusion complex)"
       class="neutral">
        SSF58038
</a>

            <span>(SNARE fus...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF58038-span-3"
      class="match superfamily uni"
      style="left:  72.174%;
              width: 19.13%;"
      title="SSF58038 250 - 315" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF58038-popup-3" style="display: none;">

        <div class="popup_topl"><span class="superfamily uni caption_puce"></span>250 - 315</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF58038' class="ext">SSF58038 </a>
            <span>(SNARE fusion complex)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:14.493%;" title="50"></span>
            <span class="grade" style="left:28.986%;" title="100"></span>
            <span class="grade" style="left:43.478%;" title="150"></span>
            <span class="grade" style="left:57.971%;" title="200"></span>
            <span class="grade" style="left:72.464%;" title="250"></span>
            <span class="grade" style="left:86.957%;" title="300"></span>
            <span class="grade" style="left:100%;" title="345"></span>

    </div>

</div>
                            </li>
                    </ol>
                    <div class="bot-row-line-bot"></div>
                </div>
            </div>


<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 0);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
</body>
</html>
