<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8" content="text/html">
    <title>
    comp6675_c0_seq0|m.28700 - InterPro
    </title>
    <meta name="description" content="InterProScan result page: Protein matches and sequence features">
    <meta name="author" content="InterPro Team, European Bioinformatics Institute">


    <link href="resources/css/type_colours.css" rel="stylesheet" type="text/css"/>
    <link class="database" href="resources/css/database.css" rel="stylesheet" type="text/css"/>
    <link href="resources/javascript/qtip2/jquery.qtip.css" rel="stylesheet" type="text/css"/>
    <link href="resources/css/protein.css" rel="stylesheet" type="text/css"/>
    <script src="resources/javascript/jquery/jquery-1.7.1.min.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery-ui-1.8.17.custom.min.js" type="text/javascript"></script>
    <script src="resources/javascript/qtip2/jquery.qtip.min.js" type="text/javascript"></script>
    <script src="resources/javascript/common.js" type="text/javascript"></script>
    <script src="resources/javascript/protein.js" type="text/javascript"></script>
    <script src="resources/javascript/protein-popups.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.cookie.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.jscroll.min.js" type="text/javascript"></script>

</head>
<body>
<div class="container_24">
<div class="grid_24 clearfix" id="content" >

    <div class="grid_5 alpha">
<div id="menu" style="display: none;">
    <div class="menu-filter">

        <h1>Filter view on</h1>

        <div class="menu-filter-type"><h1>Entry type</h1>

            <fieldset>
                <div >
                    <input type="checkbox" class="type" id="check-2" value="Family-row"
                           checked="checked" />
                    <label for="check-2" class="family">Family</label>
                </div>
                <div >
                    <input type="checkbox" class="type" id="check-3" value="Domain-row"
                           checked="checked" />
                    <label for="check-3" class="domains">Domains</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-4" value="Repeat-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-4" class="repeats">Repeats</label>
                </div>
                <div >
                    <input type="checkbox" class="type" id="check-5"
                           value="SITE-row Conserved_site-row PTM-row Binding_site-row Active_site-row"
                           checked="checked" />
                    <label for="check-5" class="site">Site</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-status"><h1>Status</h1>

            <fieldset>
                <div >
                    <input type="checkbox" name="type" id="check-6" value="unintegrated"
                           checked="checked" />
                    <label for="check-6" class="uni">Unintegrated</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-colour">
            <div class="filter-colour-head">
                <div class="filter-colour-title-l">
                    Colour by
                </div>

            </div>
            <fieldset>
                <input type="radio" name="blobColour" id="domainColour" value="domainColour" checked="checked"/>
                <label for="domainColour">domain relationship</label> <br/>

                <input type="radio" name="blobColour" id="databaseColour" value="databaseColour"/>
                <label for="databaseColour">source database</label>
            </fieldset>
        </div>
    </div>

</div>
<script type="text/javascript">
    $(document).ready(function() {
// Read colour preference from cookie (requires http://plugins.jquery.com/project/Cookie)

            // Retrieve existing cookies and set checkbox states accordingly
            var checkBoxIDs = ["#check-2", "#check-3", "#check-4", "#check-5", "#check-6"];
            for (i = 0; i < checkBoxIDs.length; i++) {
                var checkBoxId = checkBoxIDs[i];
                var cookieVal = $.cookie(checkBoxId);
                if (cookieVal != null) {
                    $(checkBoxId).prop("checked", cookieVal == "true");
                }
            }

            // and the radio button group
            var radioCookieVal = $.cookie("colour-by-domain");
            if (radioCookieVal != null) {
                if (radioCookieVal == "true") {
                    $('#domainColour').prop("checked", true);
                }
                else {
                    $('#databaseColour').prop("checked", true);
                }
            }

            // Keep the filter menu in view
//            $("#menu").jScroll();
            // CSS switching
            configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour'); // initialise

            $('input[name="blobColour"]').change(function() {
                configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour');
            });

            // Change event for type checkboxes (Family, Domain etc.)
            $(".type").change(function() {
                displayType(this);
            });

            // Initialise types
            $(".type").each(function() {
                displayType(this);
            });

            // Change event for un-integrated sig matches checkbox
            $("#check-6").change(function() {
                displayUnintegrated(this);
            });

            // Initialise un-integrated.
            // TODO Work out how to do this without each! Only one thing with an ID of "check-6"!
            $("#check-6").each(function() {
                displayUnintegrated(this);
            });

        // Make the menu visible - Javascript is enabled.
        $("#menu").css('display', 'block');
    });
</script>
    </div>

    <div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp6675_c0_seq0|m.28700 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">322 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




          <h3>Protein family membership</h3>
          <div class="Family rel_tree"><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR010255">Haem peroxidase</a><span> (IPR010255)</span><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR000823">Plant peroxidase</a><span> (IPR000823)</span></li></ul></li></ul></div>



    <h3>Domains and repeats</h3>


    <div class="prot_sum">
    <div class="bot-row">
        <div class="bot-row-line-top"></div>
        <ol class="signatures">



    <li class="signature entry-signatures">
        <div class="bot-row-signame">Domain</div>
        <div class="bot-row-line">
            <div class="matches">


<span id="supermatch-span-1"
      class="match c61 Domain"
      style="left:  7.453%;
              width: 92.547%;"
      title="Domain 25 - 322" >
</span>
        <div id="supermatch-popup-1" style="display: none;">
        <div class="popup_topl">
            <span class="c61 Domain caption_puce"></span>25 - 322
        </div>
        <div class="rel_tree"><ul><li class="Domain" ><a href="http://www.ebi.ac.uk/interpro/entry/IPR002016">Haem peroxidase, plant/fungal/bacterial</a><span> (IPR002016)</span></li></ul></div>
    </div>



                    <span class="grade" style="left:0%;" title="0"></span>
                    <span class="grade" style="left:15.528%;" title="50"></span>
                    <span class="grade" style="left:31.056%;" title="100"></span>
                    <span class="grade" style="left:46.584%;" title="150"></span>
                    <span class="grade" style="left:62.112%;" title="200"></span>
                    <span class="grade" style="left:77.64%;" title="250"></span>
                    <span class="grade" style="left:100%;" title="322"></span>

            </div>
        </div>
    </li>

        </ol>
        <div class="bot-row-line-bot"></div>
    </div>

    <div class="prot_scale">
        <div class="bot-row">

            <div class="bot-row-line">
                <div style="position:relative;">
                            <span class="scale_bar" style="left:0%;"
                                  title="1"></span>
                                <span class="scale_numb"
                                      style="left:1%;">1</span>
                            <span class="scale_bar" style="left:15.528%;"
                                  title="50"></span>
                                <span class="scale_numb"
                                      style="left:15.528%;">50</span>
                            <span class="scale_bar" style="left:31.056%;"
                                  title="100"></span>
                                <span class="scale_numb"
                                      style="left:31.056%;">100</span>
                            <span class="scale_bar" style="left:46.584%;"
                                  title="150"></span>
                                <span class="scale_numb"
                                      style="left:46.584%;">150</span>
                            <span class="scale_bar" style="left:62.112%;"
                                  title="200"></span>
                                <span class="scale_numb"
                                      style="left:62.112%;">200</span>
                            <span class="scale_bar" style="left:77.64%;"
                                  title="250"></span>
                                <span class="scale_numb"
                                      style="left:77.64%;">250</span>
                            <span class="scale_bar" style="left:100%;"
                                  title="322"></span>
                                <span class="scale_numb"
                                      style="left:100%;">322</span>
                </div>
            </div>

        </div>
    </div>

    </div>



    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Family-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_family_small.png"
                                                         alt="Family" title="Family"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR010255">IPR010255</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR010255"
                                                         class="neutral">Haem peroxidase</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR010255-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF48113" title="SSF48113 (Heme-dependent peroxidases)"
       class="neutral">
        SSF48113
</a>

            <span>(Heme-depe...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF48113-span-1"
      class="match superfamily Family"
      style="left:  7.453%;
              width: 92.547%;"
      title="SSF48113 25 - 322" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF48113-popup-1" style="display: none;">

        <div class="popup_topl"><span class="superfamily Family caption_puce"></span>25 - 322</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF48113' class="ext">SSF48113 </a>
            <span>(Heme-dependent peroxidases)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Family-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_family_small.png"
                                                         alt="Family" title="Family"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR000823">IPR000823</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR000823"
                                                         class="neutral">Plant peroxidase</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR000823-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00461 (PLPEROXIDASE)"
       class="neutral">
        PR00461
</a>

                <span>(PLPEROXIDASE)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00461-span-2"
      class="match prints Family"
      style="left:  79.193%;
              width: 5.59%;"
      title="PR00461 256 - 273" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-2" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>256 - 273</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-3"
      class="match prints Family"
      style="left:  55.901%;
              width: 4.037%;"
      title="PR00461 181 - 193" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-3" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>181 - 193</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-4"
      class="match prints Family"
      style="left:  10.559%;
              width: 6.211%;"
      title="PR00461 35 - 54" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-4" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>35 - 54</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-5"
      class="match prints Family"
      style="left:  29.503%;
              width: 4.348%;"
      title="PR00461 96 - 109" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-5" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>96 - 109</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-6"
      class="match prints Family"
      style="left:  35.404%;
              width: 3.416%;"
      title="PR00461 115 - 125" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-6" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>115 - 125</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-7"
      class="match prints Family"
      style="left:  18.012%;
              width: 6.522%;"
      title="PR00461 59 - 79" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-7" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>59 - 79</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-8"
      class="match prints Family"
      style="left:  41.304%;
              width: 4.969%;"
      title="PR00461 134 - 149" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-8" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>134 - 149</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


<span id="PR00461-span-9"
      class="match prints Family"
      style="left:  91.615%;
              width: 4.348%;"
      title="PR00461 296 - 309" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00461-popup-9" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>296 - 309</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00461&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00461 </a>
            <span>(PLPEROXIDASE)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Domain-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_domain_small.png"
                                                         alt="Domain" title="Domain"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR002016">IPR002016</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR002016"
                                                         class="neutral">Haem peroxidase, plant/fungal/bacterial</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR002016-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS50873" title="PS50873 (PEROXIDASE_4)"
       class="neutral">
        PS50873
</a>

                <span>(PEROXIDASE_4)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS50873-span-10"
      class="match prosite-profiles c61 Domain"
      style="left:  7.453%;
              width: 92.547%;"
      title="PS50873 25 - 322" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS50873-popup-10" style="display: none;">

        <div class="popup_topl"><span class="prosite-profiles c61 Domain caption_puce"></span>25 - 322</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE profiles</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS50873' class="ext">PS50873 </a>
            <span>(PEROXIDASE_4)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR002016-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00141" title="PF00141 (peroxidase)"
       class="neutral">
        PF00141
</a>

                <span>(peroxidase)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00141-span-11"
      class="match pfam c61 Domain"
      style="left:  12.733%;
              width: 75.776%;"
      title="PF00141 42 - 285" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00141-popup-11" style="display: none;">

        <div class="popup_topl"><span class="pfam c61 Domain caption_puce"></span>42 - 285</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00141' class="ext">PF00141 </a>
            <span>(peroxidase)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR002016-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00458&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00458 (PEROXIDASE)"
       class="neutral">
        PR00458
</a>

                <span>(PEROXIDASE)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00458-span-12"
      class="match prints c61 Domain"
      style="left:  41.304%;
              width: 4.037%;"
      title="PR00458 134 - 146" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00458-popup-12" style="display: none;">

        <div class="popup_topl"><span class="prints c61 Domain caption_puce"></span>134 - 146</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00458&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00458 </a>
            <span>(PEROXIDASE)</span>
        </div>
    </div>


<span id="PR00458-span-13"
      class="match prints c61 Domain"
      style="left:  74.845%;
              width: 4.969%;"
      title="PR00458 242 - 257" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00458-popup-13" style="display: none;">

        <div class="popup_topl"><span class="prints c61 Domain caption_puce"></span>242 - 257</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00458&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00458 </a>
            <span>(PEROXIDASE)</span>
        </div>
    </div>


<span id="PR00458-span-14"
      class="match prints c61 Domain"
      style="left:  56.211%;
              width: 4.969%;"
      title="PR00458 182 - 197" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00458-popup-14" style="display: none;">

        <div class="popup_topl"><span class="prints c61 Domain caption_puce"></span>182 - 197</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00458&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00458 </a>
            <span>(PEROXIDASE)</span>
        </div>
    </div>


<span id="PR00458-span-15"
      class="match prints c61 Domain"
      style="left:  17.391%;
              width: 4.658%;"
      title="PR00458 57 - 71" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00458-popup-15" style="display: none;">

        <div class="popup_topl"><span class="prints c61 Domain caption_puce"></span>57 - 71</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00458&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00458 </a>
            <span>(PEROXIDASE)</span>
        </div>
    </div>


<span id="PR00458-span-16"
      class="match prints c61 Domain"
      style="left:  35.714%;
              width: 5.59%;"
      title="PR00458 116 - 133" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00458-popup-16" style="display: none;">

        <div class="popup_topl"><span class="prints c61 Domain caption_puce"></span>116 - 133</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00458&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00458 </a>
            <span>(PEROXIDASE)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry SITE-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_site_small.png"
                                                         alt="SITE" title="SITE"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR019794">IPR019794</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR019794"
                                                         class="neutral">Peroxidase, active site</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR019794-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS00436" title="PS00436 (PEROXIDASE_2)"
       class="neutral">
        PS00436
</a>

                <span>(PEROXIDASE_2)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS00436-span-17"
      class="match prosite-patterns SITE"
      style="left:  17.391%;
              width: 3.727%;"
      title="PS00436 57 - 68" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS00436-popup-17" style="display: none;">

        <div class="popup_topl"><span class="prosite-patterns SITE caption_puce"></span>57 - 68</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE patterns</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS00436' class="ext">PS00436 </a>
            <span>(PEROXIDASE_2)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry SITE-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_site_small.png"
                                                         alt="SITE" title="SITE"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR019793">IPR019793</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR019793"
                                                         class="neutral">Peroxidases heam-ligand binding site</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR019793-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS00435" title="PS00435 (PEROXIDASE_1)"
       class="neutral">
        PS00435
</a>

                <span>(PEROXIDASE_1)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS00435-span-18"
      class="match prosite-patterns SITE"
      style="left:  56.211%;
              width: 3.416%;"
      title="PS00435 182 - 192" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS00435-popup-18" style="display: none;">

        <div class="popup_topl"><span class="prosite-patterns SITE caption_puce"></span>182 - 192</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE patterns</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS00435' class="ext">PS00435 </a>
            <span>(PEROXIDASE_1)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>


            <div class="prot_entries" id="uni_sign">
                <div class="top-row">
                    <div class="top-row-id"><img src="resources/images/ico_type_uni_small.png"
                                                 alt="Unintegrated signatures" title="Unintegrated signatures"/> no IPR
                    </div>
                    <div class="top-row-name">Unintegrated signatures</div>
                </div>
                <div class="bot-row">
                    <div class="bot-row-line-top"></div>
                    <ol class="signatures">
                            <li class="signature">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/1.10.420.10" title="G3DSA:1.10.420.10"
           class="neutral">
    G3DSA:1.10.42...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA11042010-span-19"
      class="match gene3d uni"
      style="left:  54.037%;
              width: 39.441%;"
      title="G3DSA:1.10.420.10 175 - 301" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA11042010-popup-19" style="display: none;">

        <div class="popup_topl"><span class="gene3d uni caption_puce"></span>175 - 301</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.10.420.10' class="ext">G3DSA:1.10.420.10 </a>
            <span>(G3DSA:1.10.420.10)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                            </li>
                            <li class="signature">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/1.10.520.10" title="G3DSA:1.10.520.10"
           class="neutral">
    G3DSA:1.10.52...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA11052010-span-20"
      class="match gene3d uni"
      style="left:  9.938%;
              width: 44.099%;"
      title="G3DSA:1.10.520.10 33 - 174" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA11052010-popup-20" style="display: none;">

        <div class="popup_topl"><span class="gene3d uni caption_puce"></span>33 - 174</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.10.520.10' class="ext">G3DSA:1.10.520.10 </a>
            <span>(G3DSA:1.10.520.10)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:15.528%;" title="50"></span>
            <span class="grade" style="left:31.056%;" title="100"></span>
            <span class="grade" style="left:46.584%;" title="150"></span>
            <span class="grade" style="left:62.112%;" title="200"></span>
            <span class="grade" style="left:77.64%;" title="250"></span>
            <span class="grade" style="left:100%;" title="322"></span>

    </div>

</div>
                            </li>
                    </ol>
                    <div class="bot-row-line-bot"></div>
                </div>
            </div>


<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p>
            <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006979"
                   class="ext">GO:0006979</a> response to oxidative stress
                <br/>
            
            <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114"
                   class="ext">GO:0055114</a> oxidation-reduction process
                <br/>
            </p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004601"
                   class="ext">GO:0004601</a> peroxidase activity
                <br/>
            
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037"
                   class="ext">GO:0020037</a> heme binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 1);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
</body>
</html>
