<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8" content="text/html">
    <title>
    comp14716_c0_seq0|m.43617 - InterPro
    </title>
    <meta name="description" content="InterProScan result page: Protein matches and sequence features">
    <meta name="author" content="InterPro Team, European Bioinformatics Institute">


    <link href="resources/css/type_colours.css" rel="stylesheet" type="text/css"/>
    <link class="database" href="resources/css/database.css" rel="stylesheet" type="text/css"/>
    <link href="resources/javascript/qtip2/jquery.qtip.css" rel="stylesheet" type="text/css"/>
    <link href="resources/css/protein.css" rel="stylesheet" type="text/css"/>
    <script src="resources/javascript/jquery/jquery-1.7.1.min.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery-ui-1.8.17.custom.min.js" type="text/javascript"></script>
    <script src="resources/javascript/qtip2/jquery.qtip.min.js" type="text/javascript"></script>
    <script src="resources/javascript/common.js" type="text/javascript"></script>
    <script src="resources/javascript/protein.js" type="text/javascript"></script>
    <script src="resources/javascript/protein-popups.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.cookie.js" type="text/javascript"></script>
    <script src="resources/javascript/jquery/jquery.jscroll.min.js" type="text/javascript"></script>

</head>
<body>
<div class="container_24">
<div class="grid_24 clearfix" id="content" >

    <div class="grid_5 alpha">
<div id="menu" style="display: none;">
    <div class="menu-filter">

        <h1>Filter view on</h1>

        <div class="menu-filter-type"><h1>Entry type</h1>

            <fieldset>
                <div >
                    <input type="checkbox" class="type" id="check-2" value="Family-row"
                           checked="checked" />
                    <label for="check-2" class="family">Family</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-3" value="Domain-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-3" class="domains">Domains</label>
                </div>
                <div class="disabled">
                    <input type="checkbox" class="type" id="check-4" value="Repeat-row"
                           checked="checked" disabled="disabled"/>
                    <label for="check-4" class="repeats">Repeats</label>
                </div>
                <div >
                    <input type="checkbox" class="type" id="check-5"
                           value="SITE-row Conserved_site-row PTM-row Binding_site-row Active_site-row"
                           checked="checked" />
                    <label for="check-5" class="site">Site</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-status"><h1>Status</h1>

            <fieldset>
                <div class="disabled">
                    <input type="checkbox" name="type" id="check-6" value="unintegrated"
                           checked="checked" disabled="disabled"/>
                    <label for="check-6" class="uni">Unintegrated</label>
                </div>
            </fieldset>
        </div>

        <div class="menu-filter-colour">
            <div class="filter-colour-head">
                <div class="filter-colour-title-l">
                    Colour by
                </div>

            </div>
            <fieldset>
                <input type="radio" name="blobColour" id="domainColour" value="domainColour" checked="checked"/>
                <label for="domainColour">domain relationship</label> <br/>

                <input type="radio" name="blobColour" id="databaseColour" value="databaseColour"/>
                <label for="databaseColour">source database</label>
            </fieldset>
        </div>
    </div>

</div>
<script type="text/javascript">
    $(document).ready(function() {
// Read colour preference from cookie (requires http://plugins.jquery.com/project/Cookie)

            // Retrieve existing cookies and set checkbox states accordingly
            var checkBoxIDs = ["#check-2", "#check-3", "#check-4", "#check-5", "#check-6"];
            for (i = 0; i < checkBoxIDs.length; i++) {
                var checkBoxId = checkBoxIDs[i];
                var cookieVal = $.cookie(checkBoxId);
                if (cookieVal != null) {
                    $(checkBoxId).prop("checked", cookieVal == "true");
                }
            }

            // and the radio button group
            var radioCookieVal = $.cookie("colour-by-domain");
            if (radioCookieVal != null) {
                if (radioCookieVal == "true") {
                    $('#domainColour').prop("checked", true);
                }
                else {
                    $('#databaseColour').prop("checked", true);
                }
            }

            // Keep the filter menu in view
//            $("#menu").jScroll();
            // CSS switching
            configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour'); // initialise

            $('input[name="blobColour"]').change(function() {
                configureStylesheets($('input[name="blobColour"]:checked').attr('id') == 'domainColour');
            });

            // Change event for type checkboxes (Family, Domain etc.)
            $(".type").change(function() {
                displayType(this);
            });

            // Initialise types
            $(".type").each(function() {
                displayType(this);
            });

            // Change event for un-integrated sig matches checkbox
            $("#check-6").change(function() {
                displayUnintegrated(this);
            });

            // Initialise un-integrated.
            // TODO Work out how to do this without each! Only one thing with an ID of "check-6"!
            $("#check-6").each(function() {
                displayUnintegrated(this);
            });

        // Make the menu visible - Javascript is enabled.
        $("#menu").css('display', 'block');
    });
</script>
    </div>

    <div class="grid_19 omega main-content">
        <div class="tab">
            <div class="Protein_tab">Protein</div>
        </div>

        <div class="main-box">
                <h2 class="strapline">comp14716_c0_seq0|m.43617 </h2>
<div class="prot_gal">


    <div class="prot_gal_bloc">
        <div class="prot_gal_col"><b>Length</b></div>
        <div class="prot_gal_desc">515 amino acids
        </div>
    </div>

    <br/>
    <hr/>
</div>




          <h3>Protein family membership</h3>
          <div class="Family rel_tree"><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR001128">Cytochrome P450</a><span> (IPR001128)</span><ul><li><a href="http://www.ebi.ac.uk/interpro/entry/IPR002401">Cytochrome P450, E-class, group I</a><span> (IPR002401)</span></li></ul></li></ul></div>



    <h3>Domains and repeats</h3>


    <div class="prot_sum" style="background:none;">
    <div class="bot-row">
        None predicted.
    </div>
    </div>


    <h3>Detailed signature matches</h3>


        <div class="prot_entries">
            <ol class="entries">
                    <li class="entry Family-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_family_small.png"
                                                         alt="Family" title="Family"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR001128">IPR001128</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR001128"
                                                         class="neutral">Cytochrome P450</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR001128-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00385&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00385 (P450)"
       class="neutral">
        PR00385
</a>

                <span>(P450)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00385-span-1"
      class="match prints Family"
      style="left:  70.097%;
              width: 2.33%;"
      title="PR00385 362 - 373" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00385-popup-1" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>362 - 373</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00385&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00385 </a>
            <span>(P450)</span>
        </div>
    </div>


<span id="PR00385-span-2"
      class="match prints Family"
      style="left:  59.612%;
              width: 3.495%;"
      title="PR00385 308 - 325" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00385-popup-2" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>308 - 325</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00385&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00385 </a>
            <span>(P450)</span>
        </div>
    </div>


<span id="PR00385-span-3"
      class="match prints Family"
      style="left:  85.825%;
              width: 1.942%;"
      title="PR00385 443 - 452" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00385-popup-3" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>443 - 452</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00385&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00385 </a>
            <span>(P450)</span>
        </div>
    </div>


<span id="PR00385-span-4"
      class="match prints Family"
      style="left:  87.573%;
              width: 2.33%;"
      title="PR00385 452 - 463" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00385-popup-4" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>452 - 463</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00385&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00385 </a>
            <span>(P450)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.709%;" title="50"></span>
            <span class="grade" style="left:19.417%;" title="100"></span>
            <span class="grade" style="left:29.126%;" title="150"></span>
            <span class="grade" style="left:38.835%;" title="200"></span>
            <span class="grade" style="left:48.544%;" title="250"></span>
            <span class="grade" style="left:58.252%;" title="300"></span>
            <span class="grade" style="left:67.961%;" title="350"></span>
            <span class="grade" style="left:77.67%;" title="400"></span>
            <span class="grade" style="left:87.379%;" title="450"></span>
            <span class="grade" style="left:100%;" title="515"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001128-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://pfam.sanger.ac.uk/family/PF00067" title="PF00067 (p450)"
       class="neutral">
        PF00067
</a>

                <span>(p450)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PF00067-span-5"
      class="match pfam Family"
      style="left:  6.214%;
              width: 88.35%;"
      title="PF00067 33 - 487" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PF00067-popup-5" style="display: none;">

        <div class="popup_topl"><span class="pfam Family caption_puce"></span>33 - 487</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>Pfam</b> <abbr class="icon icon-generic" data-icon="i" title="Pfam is a collection of protein family alignments which were constructed semi-automatically using hidden Markov models (HMMs). Sequences that were not covered by Pfam were clustered and aligned automatically, and are released as Pfam-B. Pfam families have permanent accession numbers and contain functional annotation and cross-references to other databases, while Pfam-B families are re-generated at each release and are unannotated."></abbr> <br/>
            <a href='http://pfam.sanger.ac.uk/family/PF00067' class="ext">PF00067 </a>
            <span>(p450)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.709%;" title="50"></span>
            <span class="grade" style="left:19.417%;" title="100"></span>
            <span class="grade" style="left:29.126%;" title="150"></span>
            <span class="grade" style="left:38.835%;" title="200"></span>
            <span class="grade" style="left:48.544%;" title="250"></span>
            <span class="grade" style="left:58.252%;" title="300"></span>
            <span class="grade" style="left:67.961%;" title="350"></span>
            <span class="grade" style="left:77.67%;" title="400"></span>
            <span class="grade" style="left:87.379%;" title="450"></span>
            <span class="grade" style="left:100%;" title="515"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001128-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
        <a href="http://www.cathdb.info/superfamily/1.10.630.10" title="G3DSA:1.10.630.10"
           class="neutral">
    G3DSA:1.10.63...
</a>


</div>
<div class="bot-row-line">
    <div class="matches">



<span id="G3DSA11063010-span-6"
      class="match gene3d Family"
      style="left:  4.854%;
              width: 88.35%;"
      title="G3DSA:1.10.630.10 26 - 480" >
</span>

    <script type="text/javascript">

    </script>

    <div id="G3DSA11063010-popup-6" style="display: none;">

        <div class="popup_topl"><span class="gene3d Family caption_puce"></span>26 - 480</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>GENE3D</b> <abbr class="icon icon-generic" data-icon="i" title="Gene3D HMMs extended predictions of CATH protein structures."></abbr> <br/>
            <a href='http://www.cathdb.info/superfamily/1.10.630.10' class="ext">G3DSA:1.10.630.10 </a>
            <span>(G3DSA:1.10.630.10)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.709%;" title="50"></span>
            <span class="grade" style="left:19.417%;" title="100"></span>
            <span class="grade" style="left:29.126%;" title="150"></span>
            <span class="grade" style="left:38.835%;" title="200"></span>
            <span class="grade" style="left:48.544%;" title="250"></span>
            <span class="grade" style="left:58.252%;" title="300"></span>
            <span class="grade" style="left:67.961%;" title="350"></span>
            <span class="grade" style="left:77.67%;" title="400"></span>
            <span class="grade" style="left:87.379%;" title="450"></span>
            <span class="grade" style="left:100%;" title="515"></span>

    </div>

</div>
                                    </li>

                                    <li id="IPR001128-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF48264" title="SSF48264 (Cytochrome P450)"
       class="neutral">
        SSF48264
</a>

            <span>(Cytochrom...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="SSF48264-span-7"
      class="match superfamily Family"
      style="left:  6.214%;
              width: 87.573%;"
      title="SSF48264 33 - 483" >
</span>

    <script type="text/javascript">

    </script>

    <div id="SSF48264-popup-7" style="display: none;">

        <div class="popup_topl"><span class="superfamily Family caption_puce"></span>33 - 483</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>SUPERFAMILY</b> <abbr class="icon icon-generic" data-icon="i" title="SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins. Models correspond to SCOP domains at the superfamily level and a hybrid method subsequently sub-classifies domains at the family level. SUPERFAMILY provides its structural assignments at both levels to all completely sequenced genomes."></abbr> <br/>
            <a href='http://supfam.cs.bris.ac.uk/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF48264' class="ext">SSF48264 </a>
            <span>(Cytochrome P450)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.709%;" title="50"></span>
            <span class="grade" style="left:19.417%;" title="100"></span>
            <span class="grade" style="left:29.126%;" title="150"></span>
            <span class="grade" style="left:38.835%;" title="200"></span>
            <span class="grade" style="left:48.544%;" title="250"></span>
            <span class="grade" style="left:58.252%;" title="300"></span>
            <span class="grade" style="left:67.961%;" title="350"></span>
            <span class="grade" style="left:77.67%;" title="400"></span>
            <span class="grade" style="left:87.379%;" title="450"></span>
            <span class="grade" style="left:100%;" title="515"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry Family-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_family_small.png"
                                                         alt="Family" title="Family"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR002401">IPR002401</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR002401"
                                                         class="neutral">Cytochrome P450, E-class, group I</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR002401-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off" title="PR00463 (EP450I)"
       class="neutral">
        PR00463
</a>

                <span>(EP450I)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PR00463-span-8"
      class="match prints Family"
      style="left:  57.476%;
              width: 3.495%;"
      title="PR00463 297 - 314" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-8" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>297 - 314</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-9"
      class="match prints Family"
      style="left:  12.039%;
              width: 3.883%;"
      title="PR00463 63 - 82" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-9" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>63 - 82</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-10"
      class="match prints Family"
      style="left:  16.699%;
              width: 4.272%;"
      title="PR00463 87 - 108" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-10" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>87 - 108</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-11"
      class="match prints Family"
      style="left:  87.573%;
              width: 4.66%;"
      title="PR00463 452 - 475" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-11" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>452 - 475</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-12"
      class="match prints Family"
      style="left:  69.903%;
              width: 3.689%;"
      title="PR00463 361 - 379" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-12" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>361 - 379</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-13"
      class="match prints Family"
      style="left:  85.631%;
              width: 2.136%;"
      title="PR00463 442 - 452" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-13" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>442 - 452</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-14"
      class="match prints Family"
      style="left:  61.359%;
              width: 5.243%;"
      title="PR00463 317 - 343" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-14" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>317 - 343</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


<span id="PR00463-span-15"
      class="match prints Family"
      style="left:  78.058%;
              width: 4.854%;"
      title="PR00463 403 - 427" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PR00463-popup-15" style="display: none;">

        <div class="popup_topl"><span class="prints Family caption_puce"></span>403 - 427</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PRINTS</b> <abbr class="icon icon-generic" data-icon="i" title="PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of OWL. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs: the database thus provides a useful adjunct to PROSITE."></abbr> <br/>
            <a href='http://www.bioinf.manchester.ac.uk/cgi-bin/dbbrowser/sprint/searchprintss.cgi?prints_accn=PR00463&amp;display_opts=Prints&amp;category=None&amp;queryform=false&amp;regexpr=off' class="ext">PR00463 </a>
            <span>(EP450I)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.709%;" title="50"></span>
            <span class="grade" style="left:19.417%;" title="100"></span>
            <span class="grade" style="left:29.126%;" title="150"></span>
            <span class="grade" style="left:38.835%;" title="200"></span>
            <span class="grade" style="left:48.544%;" title="250"></span>
            <span class="grade" style="left:58.252%;" title="300"></span>
            <span class="grade" style="left:67.961%;" title="350"></span>
            <span class="grade" style="left:77.67%;" title="400"></span>
            <span class="grade" style="left:87.379%;" title="450"></span>
            <span class="grade" style="left:100%;" title="515"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
                    <li class="entry SITE-row">

                        <div class="top-row">
                            <div class="top-row-id"><img src="resources/images/ico_type_site_small.png"
                                                         alt="SITE" title="SITE"/> <a
                                    href="http://www.ebi.ac.uk/interpro/entry/IPR017972">IPR017972</a></div>
                            <div class="top-row-name"><a href="http://www.ebi.ac.uk/interpro/entry/IPR017972"
                                                         class="neutral">Cytochrome P450, conserved site</a></div>
                        </div>

                        <div class="bot-row">
                            <div class="bot-row-line-top"></div>
                            <ol class="signatures" style="border:0px solid pink;">


                                    <li id="IPR017972-signatures" class="signature entry-signatures">
<div class="bot-row-signame">
    <a href="http://www.expasy.org/prosite/PS00086" title="PS00086 (CYTOCHROME_P450)"
       class="neutral">
        PS00086
</a>

            <span>(CYTOCHROME...)</span>

</div>
<div class="bot-row-line">
    <div class="matches">



<span id="PS00086-span-16"
      class="match prosite-patterns SITE"
      style="left:  86.214%;
              width: 1.942%;"
      title="PS00086 445 - 454" >
</span>

    <script type="text/javascript">

    </script>

    <div id="PS00086-popup-16" style="display: none;">

        <div class="popup_topl"><span class="prosite-patterns SITE caption_puce"></span>445 - 454</div>
        <div class="popup_botl" style="font-size:88%;">
            <b>PROSITE patterns</b> <abbr class="icon icon-generic" data-icon="i" title="PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. Profiles and patterns are constructed from manually edited seed alignments. PROSITE is complemented by ProRule, a collection of rules based on profiles and patterns, which increases the discriminatory power of profiles and patterns by providing additional information about functionally and/or structurally critical amino acids."></abbr> <br/>
            <a href='http://www.expasy.org/prosite/PS00086' class="ext">PS00086 </a>
            <span>(CYTOCHROME_P450)</span>
        </div>
    </div>


            <span class="grade" style="left:0%;" title="0"></span>
            <span class="grade" style="left:9.709%;" title="50"></span>
            <span class="grade" style="left:19.417%;" title="100"></span>
            <span class="grade" style="left:29.126%;" title="150"></span>
            <span class="grade" style="left:38.835%;" title="200"></span>
            <span class="grade" style="left:48.544%;" title="250"></span>
            <span class="grade" style="left:58.252%;" title="300"></span>
            <span class="grade" style="left:67.961%;" title="350"></span>
            <span class="grade" style="left:77.67%;" title="400"></span>
            <span class="grade" style="left:87.379%;" title="450"></span>
            <span class="grade" style="left:100%;" title="515"></span>

    </div>

</div>
                                    </li>
                            </ol>
                            <div class="bot-row-line-bot"></div>
                        </div>
                    </li>
            </ol>
        </div>



<h3>GO term prediction</h3>

<div class="prot_go">

    <div class="go_terms">
        <h4>Biological Process</h4>
        <div class="go_terms_box">

            <p>
            <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114"
                   class="ext">GO:0055114</a> oxidation-reduction process
                <br/>
            </p>
        </div>

        <h4>Molecular Function</h4>
        <div class="go_terms_box">

        <p>
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506"
                   class="ext">GO:0005506</a> iron ion binding
                <br/>
            
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016705"
                   class="ext">GO:0016705</a> oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
                <br/>
            
              <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037"
                   class="ext">GO:0020037</a> heme binding
                <br/>
            </p>
        </div>

        <h4>Cellular Component</h4>
        <div class="go_terms_box">

        <p></p>
                <p>None predicted.</p>
        </div>
    </div>
</div>

<script type="text/javascript">
    $(document).ready(function () {
            // Use hidden DIVs to display popups
            $('span[id*="span-"]').each(
                    function(i) {
                            preparePopup(this.id, 0);
                    }
            );
    });
</script>

        </div>
    </div>


</div>
</div>
</body>
</html>
