{% extends "layout.html" %}
{% block content %}
	<div class="about">
		<h2>Description and Motivation</h2>
		<h3 class="about_text">MarpoDB is a gene-centric database for <i>Marchantia polymorpha</i> Cam-1 strain genetic parts designed for genetic engineering and synthetic biology purposes. Our motivation to develop this database emerged from the need to handle and facilitate access to annotated sequence data in the most simple, clean and straightforward manner possible implementing some user interaction and community aspects. We would like to highlight that the database is agnostic and modular and will be released to the wider community through a github repository with setup instructions in the near-future.</h3>
		
		<h3 class="about_text"> We strongly emphasise that MarpoDB is still in a early stage of development and <u>the dataset is prone to contain mis-annotation errors</u>. Furthermore we encourage users to cross-check predictions with the ongoing community effort for sequencing Marchantia, currently at the early release stage hosted in <a href="https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Mpolymorpha_er" target="_blank">Phytozome</a>, which will become the primary source for Marchantia polymorpha reference sequences, or in <a href="http://marchantia.info/genome/index.php/Main_Page" target="_blank">marchantia.info</a>.</h3> 
		<h2>Technical details</h2>
		<p class="about_text">Datasets included in MarpoDB are derived from the <i>Marchantia polymorpha</i> Cam-1 & Cam-2 strains isolated by Prof. Jim Haseloff in Cambridge, UK.</p> 
		
		<p class="about_text">DNA and RNA data was obtained by performing 100bp paired-end Illumina sequencing and generating a de novo genome assembly using the <a href="http://jgi.doe.gov/data-and-tools/meraculous/" target="_blank">Meraculous 2.0</a> assembly software and a de novo transcriptome assembly by using <a href="https://github.com/fmaguire/Bridger_Assembler" target="_blank">Bridger Assembler</a>.</p>
			
		<p class="about_text">Downstream analyses for ORF prediction, domain and ortholog identification using <a href="https://transdecoder.github.io" target="_blank">Transdecoder</a>, <a href="https://www.ebi.ac.uk/interpro/" target="_blank">InterpoScan</a> analyses and <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">Blast</a> to a Viridiplantae filtered nr dataset.</p>
			
		<p class="about_text">Backend was developed using python libraries including <a href="http://flask.pocoo.org" target="_blank">Flask</a>, <a href="http://initd.org/psycopg/" target="_blank">Psycopg2</a> and <a href="http://biopython.org" target="_blank">Biopython</a> and database models were generated de novo using <a href="https://www.postgresql.org" target="_blank">PostgreSQL</a>. Custom scripts were generated for obtaining gene regions with mapped transcripts, interrogating the database and producing filtered results.</p>  
		
		<p class="about_text">Frontend was generated by using <a href="http://chmille4.github.io/Scribl/" target="_blank">Scribl</a>, <a href="https://github.com/calipho-sib/sequence-viewer" target="_blank">SeqViewer</a>, <a href="https://clipboardjs.com" target="_blank">Clipboard</a> and custom jQuery and Javascript code for programming site behaviour.</p>
		
		<p class="about_text">MarpoDB was developed by Bernardo Pollak (BP)*, Mihails Delmans(MD)* under the supervision of Prof. Jim Haseloff at the OpenPlant Centre in the Department of Plant Sciences of the University of Cambridge. BP performed extraction experiments and carried out high-throughput sequencing and assembled the genome and transcriptome. BP in conjunction with MD did the downstream analyses, prepared the datasets and programmed the front-end and MD developed the back-end and database models.</p>
		<p class="about_text">* Equal contribution.</p>
		
		<h2>Disclaimer</h2>
		<p class="about_text">This software is provided by the authors "as is" and any express or implied warranties, including, but not limited to, the implied warranties of merchantability and fitness for a particular purpose are disclaimed. The entire risk as to the quality and performance of the program is with you. In no event shall the authors, be liable for any direct, indirect, incidental, special, exemplary, or consequential damages (including, but not limited to, procurement of substitute goods or services; loss of data or data being rendered inaccurate; loss of use or profits; or business interruption) however caused and on any theory of liability, whether in contract, strict liability, or tort (including negligence or otherwise) arising in any way out of the use of this software, even if advised of the possibility of such damage.<br \><br \></p>
	</div>
{% endblock %}